<!DOCTYPE html>

<html lang="en">
    <head>
        <title>Results</title>
    </head>
    <body>
        <script src="./scripts/wiki.js"></script>
        <!-- <script src="/Template:Ionis_Paris/scripts/wiki?action=raw&ctype=text/javascript"></script> -->

        <div class="navigation"></div>
        <div class="firstView">
            <div class="container firstViewTitle">Results</div>  
        </div>
        <div></div>
        <div class="container">
                <div class="textContent mt-16 addPadding">
                    <p>
                        The aim of our project BacTail is to create tailor-made and engineered bacteria to specifically recognize, 
                        bind and lyse  pathogenic bacteria, then self-destroy, using the powerful tools of synthetic biology. For 
                        the experimental approach, we divided our project into 3 distinct axes for for the laboratory: 
                    </p>
                </div>
                <div class="containerCard" x-data="{}">
                    <div class="Card" target-id="phase1">
                        <img :src="window.LINKS.designProjectPhase1.url" class="imgCard" />
                        <div class="textCard1 projectCard"><b>Phase 1</b><br>
                            Target and bind to the bacteria of interest
                            </div>
                    </div>
                    <div class="Card" target-id="phase2">
                        <img :src="window.LINKS.designProjectPhase2.url" class="imgCard" />
                        <div class="textCard2 projectCard"><b>Phase 2</b><br>
                            Lyse the target bacteria using antimicrobial peptides
                        </div>

                    </div>
                    <div class="Card" target-id="phase3">
                        <img :src="window.LINKS.designProjectPhase3.url" class="imgCard" />
                        <div class="textCard2 projectCard"><b>Phase 3</b><br>
                            Incorporate of a Kill-switch system in our chassis to induce its self-destruction
                        </div>

                    </div>
                </div>
                <div class="check mt-16">
                    <i class="checkIcon" data-feather="check-circle"></i>
                    <div>
                        <b>The properties of our experimental approach are as follows:</b>
                    </div>
                </div>
                <div class="textContent mt-4 ml-16 addPadding">
                    <p>
                        All the steps cited above have been developed as one unique system adapted to our project – yet, 
                        these steps can be used individually and eventually be re-adapted for other application areas.
                    </p>
                </div>
                <div class="textContent mt-4 ml-16 addPadding">
                    <p>
                        All our phases are designed for <b><i>Escherichia coli K12</i></b> since we are working under <b>Class 1 Biosafety level</b>. Yet, 
                        the ultimate goal is to create a template of this system that can be specifically applied against different 
                        bacterial strains. Specific bacteriophages have already been identified to target pathogenic bacteria such as 
                        <b><i>Salmonella</i></b>, <b><i>Listeria</i></b>, <b><i>Mycobacterium tuberculosis</i></b>, <b><i>Pseudomonas</i></b> or <b><i>Escherichia coli</i></b>. For example, the PP01 phage 
                        was shown to target the shiga-toxin producing <b><i>E. coli</i></b> strain<sup>1</sup>.

                    </p>
                </div>  
                <div class="textContent mt-4 ml-16 addPadding">
                    <p>
                        Therefore, we created a software to specifically select and customize a BioBrick to recognize 
                        the desired target bacteria for phase 1.
                    </p>
                </div> 
                <div class="textContent mt-4 addPadding">
                    <p>
                        Further detail on our experimental procedures and protocols can be found here: Procedures and Protocols.

                    </p>
                </div>
                <div class="resultTitle mt-16" id="phase1">
                    Phase 1. Targeting and binding to the target bacteria
                </div> 
                <div class="resultTitle1 mt-16">
                    BioBrick and primer design of the fragments for phase 1
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The gene product 37 (gp37) is a critical component for the phase I of the project BacTail. To find out more about the 
                        scientific background and design of Phase I click here (mettre lien vers Phase I Design).<br><br>

                        The aim for the first phase is to express the gp37 on the surface of our chassis Escherichia coli  (<i>E.coli</i>) K12 thanks 
                        to the co-expressed genes Lpp-OmpA. The following BioBrick (<b>Figure 1</b>) was created and divided into 4 different fragments 
                        to ease the assembly step with the plasmid – for clarity, the fragments are summarized in <b>Figure 2B</b>.             
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results1.url" />
                <div class="textContent  mt-6 note">
                    <b>Figure 1: DNA sequence of BioBrick 1 of phase 1</b> | All the parts to express gp37 on the surface of our chassis: Promoter (BBa_J23104), Ribosome binding site (RBS - BBa_B0030),  Co-expressed lipoprotein 
                    with the outer membrane protein A (Lpp-OmpA - BBa_K1694002), Short linker (Gly-Gly-Ser-Gly) to  fuse the 
                    proteins (BBa_K243004), TEV cleavage site (BBa_J18918), 10xHis-tag (BBa_K844000), Red fluorescent protein 
                    (RFP - BBa_E1010), Gene product (gp) 37 (BBa_K3409000), Double-terminator (BBa_B0015), Promoter medium 
                    (BBa_J23107), Ribosome binding site (RBS - BBa_B0032), chaperone protein gp38 (BBa_K3409001), chaperone 
                    protein 57A (BBa_K3409002), Terminator (BBa_B1005) | Created with Benchling
                </div>    
                 <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results2.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 2: BioBrick 1 divided into four distinct fragments | A.</b> The expected assembled plasmid with Fragments 1,2,3,4 and the 
                       backbone | <b>B.</b> A table giving the detailed composition of the four fragments of the first phase of BacTail.  
                </div>
                <div class="resultTitle1 mt-16">
                    Polymer chain reaction (PCR) & gel migrations
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Once received, the synthetized fragments are resuspended and amplified via polymerase chain reaction (PCR) to 
                        obtain a higher quantity of DNA. Every PCR sample is verified by gel electrophoresis to analyze the PCR results. 
                        The first time PCR has been performed (<b>Figure 3A</b>) for Fragment (Frag.) 1, Frag. 2, Frag. 3 and Frag. 4, no bands 
                        are observable. After some troubleshooting and verifications, new primers need to be ordered to amplify 
                        Fragments 1, 2, 3, and 4 again (<b>Figure 3B</b>). However, Fragment 1 required to be amplified in two steps 
                        (<b>Figure 3B</b> – Frag.1 & Frag. 1 Bis). Due to its complexity, Fragment 1 is synthetized again. Eventually, with 
                        the new version of Fragment 1 and the optimized primers the appropriate bands for Frag. 1 (1218 bp), Frag. 2 
                        (1541 bp), Frag. 3 (1669 bp) and Frag. 4 (845 bp) are at the expected size (<b>Figure 3C</b>).
            
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results3.url" />
                 <div class="textContent mt-6 note">
                    <b>Figure 3: Gel migration results of Fragment 1,2,3, and 4 with optimization. | A.</b> 1kb+ DNA ladder was used for our gels <b>A.</b> First PCR amplification 
                    of Fragment 1,2,3 and 4. <b>B.</b> Second round of PCR amplification of Fragment 1,2,3, and 4 <b>C.</b> 
                    Third and last round with optimized conditions of PCR amplification of Fragment 1,2,3, and 4 <b>D.</b> Expected results 
                    generated after PCR simulation with SnapGene.
                </div>
                <div class="resultTitle1 mt-16">
                    Assembly step
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The four fragments purified in the previous step (see above) sequences have been assembled into the plasmid 
                        pSB1A3 using the NEBuilder Hifi Dna Assembly Master Mix. This technique is a molecular cloning method used to join 
                        multiple DNA fragments in a single, isothermal reaction2. The fragments were designed in such a way that adjacent 
                        fragments have identical sequences on the ends, called overhangs, see <b>Figure 4</b>. Overhangs are added during the synthesis 
                        of the different fragments. However, overhang sequences can be added via PCR with primers containing a 5‘ end identical to 
                        the adjacent fragment and a 3‘ end that anneals to the target sequence. In our case, the primers are 30 base pairs (bp) 
                        long, where the first 15 bp match the end of the adjacent fragment and the last 15 bp annealed to the target sequence. 
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results4.url" />
                 <div class="textContent mt-6 note">
                    <b>Figure 4: Schema of the HiFi Assembly reaction</b> | Created with BioRender.com

                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results5.url" />
                 <div class="textContent mt-6 note">
                    <b>Figure 5: Expected results of the Hifi Assembly of the 4 fragments and the plasmid pSB1A3 | A.</b> The expected assembled plasmid with Fragments 1,2,3,4 and the 
                        backbone | created with SnapGene
                </div>
                <div class="resultTitle1 mt-16">
                    Conclusion
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Due to difficulties encountered during the experimental procedures and time limitation, we decided to change 
                        our verification technique and ordered a new DNA fragment. More information on this in Engineering Success.
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    New BioBrick – Version 2: without RFP
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Due to difficulties encountered during the experimental procedures and time limitation, we decided to change 
                        our verification technique and ordered a new DNA fragment. More information on this in Engineering Success.
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results6.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 6: New  DNA sequence of BioBrick 1 version 2 (V2) of phase 1 |</b> All the parts to express gp37 on the surface of our chassis: Promoter (BBa_J23104), Ribosome binding site (RBS - BBa_B0030),  Co-expressed lipoprotein with the outer 
                    membrane protein A (Lpp-OmpA - BBa_K1694002), Short linker (Gly-Gly-Ser-Gly) to  fuse the proteins (BBa_K243004), 
                    TEV cleavage site (BBa_J18918), 10xHis-tag (BBa_K844000), Gene product (gp) 37 (BBa_K3409000), Double-terminator 
                    (BBa_B0015),Promoter medium (BBa_J23107), Ribosome binding site (RBS - BBa_B0032), chaperone protein gp38 (BBa_K3409001),
                     chaperone protein 57A (BBa_K3409002), Terminator (BBa_B1005) | Created with Benchling
                </div>
                <div class="resultTitle1 mt-16">
                    Revelation of the presence of gp37 via immunofluorescence labelling
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The aim was to detect the receptor binding protein gp37 expressed and directed to the membrane thanks to the co-expressed 
                        Lpp-OmpA genes. This is achieved using fused His-tag placed after the OmpA and before the gp37. Thus, the His-tag  was 
                        targeted with a primary monoclonal anti-his antibody conjugated to a fluorescence marker, fluorescein isothiocyanate (FITC), 
                        which excitation and emission spectrum peak has a wavelengths of approximately 495 nm and 519 nm<sup>6</sup>, detectable via fluorescence 
                        spectrophotometry. 
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results7.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 7: Expected results of the immunofluorescence labelling | 1st possibility:</b> In the case of binding of the monoclonal anti-his 
                    antibody to the his-tag, a signal is detected at 519 nm. <b>2nd  possibility:</b> No attachment of the antibody and therefore no signal is 
                    detected at 519 nm | Created with BioRender.com
                </div>
                <div class="resultTitle1 mt-16">
                    Experimental perspectives Phase 1 | Targeting and binding to the target bacteria
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        <ul>
                        <li>The presence of the receptor binding protein on the surface of our chassis should be confirmed through repetitions of immunolabeling experiments.<br><br></li>

                        <li>Perform the experiments with the TEV protease and proceed with the protein purification Ni2+ chromatography columns specific to the his-tag.<br><br></li>

                        <li>Validate experiments with confocal microscopy to have the possibility to see any signal at the surface of the chassis.<br><br></li>

                        <li>Confirm the binding of the chassis to the target.</li>
                    </ul>
                    </p>
                </div>    
                <div class="resultTitle mt-16" id="phase2">
                    Phase 2. Lysis of the target bacteria of interest 
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The aim of this phase is to lyse specific target bacteria. Thus, the microcin MccPDI (Proximity Dependent Inhibition) is the critical 
                        component in phase 2. The MccPDI gene cluster has been identified in the strain Escherichia coli (E.coli) 25 plasmid (Eberhart et al., 2012 ) 
                        for its ability to kill several strains of E. coli. MccPDI targets pathogenic strains as well as Biosafety Level 1 E. coli strains. To find 
                        out more about the scientific background and design of Phase II please read: (mettre lien vers Phase II Design).
                    </p>
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        For the sake of our project BacTail, it was the necessary to express accurate genes in order to secrete the antimicrobial peptide to specifically 
                        lyse the target bacterium. For the construction of our BioBrick, we used the MccPDI gene cluster because its genes are all necessary for the appropriate 
                        expression, activity, resistance, and secretion of microcin PDI.
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    BioBrick and primer design of phase II BioBricks 1,2, and 3
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The organization of the native MccPDI gene cluster found in E. coli 25 helped us design our own BioBricks to 
                        enable the engineered bacteria to lyse the bacteria of interest, illustrated below : 
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results8.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 8: Our constructs for the expression and secretion of McpM | BioBrick 1</b> (BBa_K3409010) contains the gene mcpM which when expressed, 
                    will lead microcin PDI. This microcin has two N-Ter signal peptides which are cleaved when secreted.  The gene mcpA also present in this 
                    BioBrick encodes for an activator of McpM and contains a CaaX protease domain. Finally, the gene mcpI encodes for the immunity protein of 
                    McpM for the producing strain; <b>BioBrick 2</b> (BBa_K3409011) is coding for the McpD protein which acts as a membrane fusion protein creating a 
                    channel through the periplasm and connects to TolC; <b>BioBrick 3</b> (BBa_K3409012) is coding for the McpB protein which contains transmembrane 
                    domains as well as nucleotide binding domains which have highly conserved motifs characteristic of the ABC transporter superfamily of 
                    proteins | Created with Benchling.

                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results9.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 9: The final construct for the expression and secretion of McpM | A.</b> The expected assembled plasmid with BioBricks 1,2,3 and the backbone | <b>B.</b> A 
                    table giving the detailed composition of the three BioBricks of the second phase of BacTail.  
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Concerning the design, the three BioBricks were synthetized for further assembly via IDT. Thus, we decided to add 
                        homologous sequences by PCR between the fragments to assemble for <b>Hifi Assembly</b> method (see Hifi Assembly in the Assembly 
                        section further down this the page). Additionally, we added BioBrick prefix and suffix enzyme restriction sites to assemble 
                        the entire cluster via the <b>BioBrick Assembly</b> method as well (see BioBrick Assembly in the Assembly section further down this the page).

                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results10.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 10: Schematic representation of the three BioBricks for phase II and their corresponding length (bp) | A.</b> Primers for the BioBrick 
                        Assembly method amplifying on the prefix and suffix region <b>B.</b> Primers for the Hifi Assembly method, the primers bind after the prefix and 
                        suffix region and add a homologous short sequence of the fragment it will be assembled with | Created with BioRender.com
                </div>
                <div class="resultTitle1 mt-16">
                    Polymer chain reaction (PCR) and gel migration of BioBrick 1, BioBrick 2, and BioBrick 3
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Once received, the synthetized fragments are resuspended and amplified via polymerase chain reaction (PCR) to obtain a higher quantity of DNA. 
                        Every PCR sample is verified by gel electrophoresis to analyze the PCR results. First, the appropriate bands for BioBrick 1 (1.5 kb), BioBrick 2 (1.4 kb), 
                        and BioBrick 3 (2.4 kb) amplified with the primers for Hifi Assembly method are present at their expected size (<b>Figure 5A</b>). Then, the appropriate bands for 
                        BioBrick 1 (1.5 kb), BioBrick 2 (1.4 kb), and BioBrick 3 (2.4 kb) amplified with the primers for BioBrick Assembly method, are present at their expected size 
                        on <b>Figure 5B</b>. 
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results11.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 11: Gel migration results of BioBricks 1, 2 and 3 |</b> A 1kb+ DNA ladder was used for all gels A. PCR amplification of BioBrick (BB) 1(1.5 kb) , BioBrick 2 (1.4 kb) and 
                    BioBrick 3 (2.4 kb) with primers for Hifi Assembly method B. PCR amplification of BioBrick (BB) 1(1.5 kb), BioBrick 2 (1.4 kb) and BioBrick 3 (2.4 kb) with primers for BioBrick 
                    Assembly method C. Expected results simulated with SnapGene containing the 1 kb+ DNA ladder, BioBrick (BB) 1(1.5 kb) , BioBrick  2 (1.4 kb) and BioBrick 3 (2.4 kb).
                </div>
                <div class="resultTitle1 mt-16">
                    Assembly step
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The three fragments purified in the previous step (see above) are assembled into the plasmid pSB1A3, pSB4A5 or pSB4K5 either using Hifi 
                        Assembly or restriction enzyme (RE) digestion and ligation.  
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    A. Hifi DNA Assembly method  
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Hifi DNA Assembly method is a molecular cloning technique used to join multiple DNA fragments in a single isothermal reaction2. As depicted in 
                        <b>Figure 4B</b>, the three BioBrick fragments were designed to add identical end sequences, or overhangs to adjacent fragment with specific 
                        primers via PCR.
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results12.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 12: Hifi Assembly Method of BioBricks 1,2 and 3 |</b> 1. BioBrick fragments and backbone are put together for assembly; 2. The end results is the assembled 
                    circular vector containing all the BioBricks and plasmid backbone; 3. Assembled product is transformed into competent cells | Created with BioRender.com
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results13.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 13: Transformation after Hifi assembly of Phase 2 – BioBrick 1,2,3 and pSB1A3 | Plate on the left:</b> Escherichia coli DH5 alpha transformed with Hifi assembled product 
                    of Phase 2 (BioBrick 1,2,3 and pSB1A3). Plate on the right: Escherichia coli DH5 alpha transformed with the Hifi Assembly positive control.
                </div>
                <div class="resultTitle1 mt-16">
                    Conclusion  
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        After Hifi Assembly for phase 2, we should obtain a plate with a certain number of white colonies. However, all the colonies we can 
                        see on the plate on the left of Figure 6 are red and outnumbered by small colonies present on the positive control,  which can be observed 
                        on right plate of Figure 6. These results show that the Hifi Assembly for the phase 2 did not worked. The red color of the colonies suggests 
                        that the cells were transformed with the plasmid still containing the RFP gene yet not the inserts. This experiment has been repeated 3 times 
                        with no concluding outcome. The second strategy has been carried out for the assembly, although it is more time-consuming .                        
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    B. Restriction enzyme (RE) digestion & ligation assembly method
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Different workflows have been tested in parallel for the BioBrick Assembly method. One is the <b>2-step workflow</b> where the the ligation is performed 
                        in 2 steps as depicted in <b>Figure 8</b>. The other one is the <b>1-step workflow</b> where the ligation is in done in one step.                        
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    The 2-step workflow
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        BioBrick 2 and 3 need to be assembled first since BioBrick 1 contains the McpM which is toxic for bacteria (see Design Phase 2). Thus, BioBrick 2 and 3 are not 
                        killing bacteria during their transformation. As a test, BioBrick 1 is inserted into pSB1A3 and transformed into competent cells, see <b>Figure 7</b>.                        
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results14.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 14: DH5 alpha bacteria transformed with BioBrick 1 in pSB1A3 |</b> Plates on the left are the controls (Bacteria transformed only with the plasmid incubated at 30°C and 37°C) 
                    and plates on the right are bacteria that have been transformed with the ligated product (plasmid + BioBrick 1 incubated at 30°C and 37°C).
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Concerning the test of the insertion of BioBrick 1 into the pSB1A3, plates have been incubated at 30°C and 37°C degrees. Both plates with the 
                        sample (Figure 6 – on the right) containing only BioBrick 1 do not show any colony. On the other side, the control plates (Figure 6 – on the left) 
                        displayed a few colonies. This suggests that bacteria was not able to survive when expressing McpM in BioBrick 1. McpM thus seems to be toxic for 
                        the bacteria.  
                    </p>
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        To assemble BioBrick 2, 3, and the backbone, BioBrick 2 is digested with EcorI and SpeI, BioBrick 3 is digested with XbaI and PstI, and the backbone with 
                        EcorI and PstI. The cohesive ends of the cut SpeI and XbaI sites can ligate. This resulted in the incorporation of the BioBrick scar sequence (5' TACTAGAG 3’) 
                        between BioBrick 2 and 3.  
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results15.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 15: 2-step BioBrick Assembly method of BioBricks 1,2 and 3 |</b> 1. Restriction (RE) digestion: BB2 with EcoRI and SpeI, BB3 with XbaI and PstI, backbone with EcorI and PstI; 2. Ligation; 3. 
                    Bacterial transformation with the ligated product; 4. Miniprep of the transformed bacteria to collect the plasmid; 5. RE digestion of the plasmid construction with EcorI and XbaI and BB1 EcorI and SpeI;  
                    6. Ligation;  7. Bacterial transformation with the ligated product | Created with BioRender.com 
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The first ligation product of the <b>2-step workflow</b> which is BioBrick 2, BioBrick 3, and the plasmid pSB1A3 is validated by colony PCR and RE digestion.
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    Results of the 2-step workflow 
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results16.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 7: Expected results of the immunofluorescence labelling | 1st possibility:</b> In the case of binding of the monoclonal anti-his 
                    antibody to the his-tag, a signal is detected at 519 nm. <b>2nd  possibility:</b> No attachment of the antibody and therefore no signal is 
                    detected at 519 nm | Created with BioRender.com
                </div>
                <div class="resultTitle1 mt-16">
                    Conclusion of the 2-step workflow 
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        When compared to different ligation ratios, a proportional amount of white colonies can be observed on the plates (Figure 9A) with no colonies counted on the 
                        negative control plate. Thus, the transformation seems to have worked. To verify the successful ligation of the plasmid with BB2 and BB3, colonies are tested 
                        after miniprep DNA isolation to purify the plasmid of interest. The plasmids purified from the bacterial colonies have been cut with restriction enzymes (RE) 
                        BgII and NdeI. The expected bands are depicted on <b>Figure 9C</b>. Sample of lane 2, 4, and 6 , corresponding to colony N°1, colony N°3, and colony N°5 respectively,
                        seem to have bands at 3.5 kb and 2.4 kb which correspond to the expected size.<br><br>

                        This workflow was stopped at this step for a matter of time. However, the next step would have been to assembly the obtained plasmid of colonies 1, 3, and 5 with 
                        the BioBrick and after confirmation to continue with the verification procedures (Via agar diffusion assays among others).
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    The 1-step workflow
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        To assemble BioBrick 1, BioBrick 2, BioBrick 3, and the backbone, BioBrick 1 is digested with EcoRI and SpeI, BioBrick 2 is digested with XbaI and SpeI, BioBrick 3 
                        is digested with XbaI and PstI, and the backbone with EcorI and PstI. The cohesive ends of the cut SpeI and XbaI sites can ligate together and as a result a BioBrick 
                        scar sequence (5' TACTAGAG 3’) will be between both BioBrick 1 and 2, and BioBrick 2 and 3. 
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results17.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 17: 1-step BioBrick Assembly method of BioBricks 1,2 and 3 | </b>All in the same tube: 1. 1st Ligation with BioBrick 1 and BioBrick 2; 2. Addition of BioBrick 3 for 2nd 
                    Ligation; Addition of the plasmid for 3rd ligation. Assembled plasmid can be transformed| Created with BioRender.com 

                </div>
                <div class="resultTitle1 mt-16">
                    Results of the 1-step workflow
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results18.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 18: 1-step workflow - Transformation of assembled BB1, BB2, BB3 and pSB1A3 | A. DH5 alpha plates with ligated product of BB1, BB2 BB2 and pSB1A3, and the control:</b> 
                    Plate N°1 One-step workflow ligated product with plasmid 39; Plate N°2 One-step workflow ligated product with plasmid 31 <b>B. Gel migration of the RE digestions: Well N°1,2,3</b>: 
                    1kb DNA Ladder // <b>Well N°2, 3, 4</b>: Colony C1, C2, C3 of plasmid 39 Phase II (miniprep) digested by  BglI // Well N°5 -> 14 : Colony 1 ->10 plasmid 39 Phase II (miniprep) 
                    digested by BglI // <b>Well N°15, 16, 17, 18</b>: Colony C1, C2, C3, C4 of plasmid 31 phase II (miniprep) digested by NdeI // <b>Well N°19, 20, 21, 22</b>: Colony C1, C2, C3, C4 of 
                    plasmid 31 phase II (miniprep) digested by SapI <b>C. Expected resultsimulated with SnapGene</b>: Well N°MW : 1kb+ DNA Ladder //Well N°1: plasmid 39 Phase II digested by BglI 
                    expected //Well N°2: plasmid 31 Phase II digested by SapI  expected.
                </div>
                <div class="resultTitle1 mt-16">
                    Conclusion of the 1-step workflow
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        Several white colonies are obtained on plate N°1 and N°2 (<b>Figure 11A</b>) and no colonies are counted on the control (plate not present). Thus, 
                        the transformation seems to have worked. Regarding the verification of the ligation, colonies needed to be tested after having completed a 
                        miniprep procedure to obtain the plasmid. The extracted plasmids of the colonies have been cut with restriction enzymes (RE) other than the 
                        one used for the BioBrick Assembly method. Then, NdeI is used to linearize the plasmid 31. SapI is used to cut twice plasmid 31 to obtain 2 
                        distinct bands at 10 kb and 3.3 kb as depicted on Well N°2 <b>Figure 11C</b>. For plasmid 39 we did not have any RE available in our inventory to 
                        linearize it. Thus, we only used BglI to cut twice plasmid 39 to obtain 2 distinct bands at 6.7 kb and 1.6 kb as depicted on Well N°1 <b>Figure 11C</b>.

                        <b>For colony 3 of plasmid 39</b>: On lane 4, three bands can be observed: one around 6.7 kb and one around 1.6 kb and a last one unexpected around 3.5 kb. 
                        The two first bands correspond to the expected results. 

                        Nevertheless, we proceeded with the verification step for the colony 3 of plasmid 39.
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    Verification of the McpM expression 
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The aim of this phase is to lyse specific target bacteria through the expression of the microcin McpM. After the assembly of the three BioBricks, the presence of 
                        McpM in the introduced plasmid need to be confirmed. 
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                    A. Extraction, purification and validation of McpM production
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        To validate the production of McpM, the protein need to be extracted and purified from the cellular culture via Ni2+ chromatography column and further revelation via SDS-Page
                        and staining technique.  
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results19.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 19: Extraction, purification and validation of McpM production |</b> 1. Transformation of the competent bacteria with our final construction containing the McpM and its 
                    secretion system; 2. Overnight growth of the cells; 3. Overnight inoculation of the appropriate clone; 4. Cell lysis; 5. Purification with the his-tag with a Ni2+ IMAC 
                    column; 6. SDS-Page, analysis of purification via staining  | Created with BioRender.com
                </div>
                <div class="resultTitle1 mt-16">
                    Expected results
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The McpM size is 8.2 kDa. Therefore, we expect that in the lane of the purified cell lysate (elutions) a band appears at the size of 8.2 kDa. 
                        In the first lane, this band should be more intense than lanes where Elution 2 and 3 have been loaded. We also expect to have few bands in the 
                        flow-through, corresponding to potential waste material. Regarding the lysate, the same bands as the flow through might appear, plus one at 8.2 kDa 
                        corresponding to McpM. Finally, in the lane of the Laemmli no bands should appear.
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results20.url" />
                <div class="textContent  mt-6 note">
                    <b>Figure 20: Expected results of extraction, purification and validation of McpM production |</b> Well 1: Ladder, Well 2: Elution tube N°1, Well 3: Elution tube N°2, 
                    Well 4: Elution tube N°3, Well 5: Flow through, Well 6: Total lysate, Well 7: Laemmli

               </div>
                <div class="resultTitle1 mt-16">
                    B. Agar diffusion assay
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results21.url" />
                 <div class="textContent  mt-6 note">
                    <b>Figure 21: Agar diffusion assay to validate the secretion of McpM | 1. The day before:</b> Overnight inoculation of the Escherichia coli (E.coli) transformed only with the 
                    pSB1A3, E.coli DH5 alpha transformed with our construction (BB1,BB2,BB3), E.coli DH10B containing the native MccpDI cluster. <b>2. The day of the experiment:</b> The E.coli only 
                    with the pSB1A3 are lawn on the plate (spread culture onto the agar) and 5-10 ul of supernatant of E.coli DH10B, E.coli DH5 alpha (BB1,BB2,BB3) and LB only are added on 6-mm 
                    ter discs on the plate and placed at 37°C overnight. <b>3. The day after:</b> A little circle should appear around the disc of the cultures that expressed 
                    mcpM | Created with BioRender.com
                </div>
                <div class="resultTitle1 mt-16">
                    Expected results
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        The E.coli DH5 alpha (BB1,BB2,BB3) should kill the surrounding plated E.coli DH5 alpha when it expresses McpM. Therefore, they are forming little radiuses
                         around their colony of the white 6-mm filter discs. This observation confirms their ability to kill E.coli DH5 alpha bacteria. This antimicrobial activity 
                         is what we are aiming for phase 2 of the BacTail project. 
                    </p>
                </div>
                <div class="resultTitle1 mt-16">
                  Results
                </div>
                <div class="textContent mt-4 addPadding">
                    <p>
                        We tested in duplicate colony with plasmid 39 assembled with BioBrick 1,2, and 3 (Disc N°2 and N°4), in duplicate colony with plasmid 31 assembled with BioBrick 1,2, 
                        and 3 (Disc N°1 and 3), in duplicate E.coli DH10B strain containing the native MccpDI cluster (Disc N°5 and 7) and as a control only LB medium (Disc 6).  
                    </p>
                </div>
                <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results22.url" />
                <div class="resultTitle1 mt-16">
                    Experimental perspectives of Phase 2| The lysis of the target bacteria  
                  </div>
                  <div class="textContent mt-4 addPadding">
                    <p>
                        We should continue with the 2-step workflow to add BioBrick 1 to the BioBrick 2, 3 assembled fragments with the plasmid and transform it into competent cells. After 
                        transformation, the plasmid needs to be purified to proceed with the verification of the assembly of BB1, BB2, and BB3, which means digestion of the plasmid with an 
                        enzyme that cuts it once to linearize it and another pair of enzymes that cut it twice in the region on the insert. Additionally, the plasmid should be cut with XbaI 
                        and PstI to perform an insert release. Furthermore, a colony PCR can be performed. After having verified that the plasmid was well assembled, the selected clone will 
                        continue with the further verification steps. These steps include the extraction, purification, and validation of McpM production via Ni2+ chromatography columns and 
                        SDS-Page. Finally, an agar diffusion assay should be performed as well. 
                    </p>
                </div>

        </div>
        <div class="references">
            <div class="container">
                <hr />
                <div class="referencesText mb-6">

                    PHASE1<br><br>
                    [1] Yoichi, M., Abe, M., Miyanaga, K., Unno, H., & Tanji, Y. (2005). Alteration of tail fiberprotein gp38 enables T2 phage to 
                    infect Escherichia coli O157: H7. Journal ofbiotechnology, 115(1), 101-107.<br><br>

                    [2] Cas, S. DNA Assembly & Synthetic Biology.<br>
                    https://cdn-api.swapcard.com/public/files/2326aec54f1043499e31fdfc78562e2d.pdf<br><br>

                    [3] Waugh D. S. (2011). An overview of enzymatic reagents for the removal of affinity tags. Protein 
                    expression and purification, 80(2), 283–293.<br>
                    https://doi.org/10.1016/j.pep.2011.08.005<br><br>

                    [4] Francisco, J. A., Campbell, R., Iverson, B. L., & Georgiou, G. (1993). Production and fluorescence-activated 
                    cell sorting of Escherichia coli expressing a functional antibody fragment on the external surface. 
                    Proceedings of the National Academy of Sciences of the United States of America, 90(22), 10444–10448.<br> 
                    https://doi.org/10.1073/pnas.90.22.10444<br><br>

                    [5] Nie, D., Hu, Y., Chen, Z. et al. Outer membrane protein A (OmpA) as a potential therapeutic target 
                    for Acinetobacter baumannii infection. J Biomed Sci 27, 26 (2020).<br>
                    https://doi.org/10.1186/s12929-020-0617-7<br><br>

                    [6] The, T. H., & Feltkamp, T. E. (1970). Conjugation of fluorescein isothiocyanate to antibodies. I. 
                    Experiments on the conditions of conjugation. Immunology, 18(6), 865–873.<br><br>

                    
                </div>
            </div>
        </div>
        <footer></footer>
    </body>
    <style>
        html,
        body {
            background-color: #fafafa !important;
        }

        .firstViewTitle {
            color: #8d1f24;
            text-shadow: 0px 10px 20px rgba(141, 31, 37, 0.3);
            font-size: 50px;
            text-transform: uppercase;
            font-weight: 800;
            height: 150px;
            display: flex;
            align-items: flex-end;
        }
        
        .textContent {
            max-width: 900px;
            font-size: 15px;
            
        }

        .textContent p {
            margin-bottom: 5px;
        }

        .textContent p:last-child {
            margin-bottom: 0;
        }

        .containerCard {
            display: flex;
            gap: 30px;
            margin-top: 50px;
        }

        .Card {
            background-color: white;
            border-radius: 20px;
            -webkit-box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            -moz-box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            border: 1px solid rgba(0, 0, 0, 0.03);
            height: 300px;
            max-width: 250px;
            flex-grow: 1;
            opacity: 1;
            position: relative;
            display: flex;
            justify-content: center;
            align-items: center;
            transform: scale(1);
        }

        .textCard1 {
            position: absolute;
            bottom: 30px;
            margin:10px;
            text-align: center;
            color: #8d1f24;
            
        }

        .textCard2 {
            position: absolute;
            bottom: 5px;
            margin:10px;
            text-align: center;
            color: #8d1f24;
            
        }

        .imgCard{
            position: absolute;
            bottom: 120px;
            height: 150px;
        }

        .Card:hover{
            transform: scale(1.1);
        }

        .check {
            display: flex;
            align-items: center;
            color: #8d1f24;
            
        }

        .checkIcon {
            height: 40px;
            width: 40px;
            margin-right: 30px;
            color: #8d1f24;
            
        }

        .resultTitle {
            color: #8d1f24;
            background-color: rgba(141, 31, 37, 0.1);
            display: inline-block;
            font-weight: 800;
            font-size: 25px;
            line-height: 1;
            padding: 10px 20px;
            border-radius: 10px;
            margin-top: 80px;
        }
        
        .resultTitle1 {
            color: #8d1f24;
            font-size: 15px;
            font-weight: 800;
        }
       
        .imgColiKaze {
            max-width: 900px;
            width: 100%;
            background-color: white;
            border-radius: 15px;
            padding: 10px 10px;
        }

        .note {     
            font-size: 13px;
            
        }

     

        .references {
            margin-top: 60px;
        }

        .referencesText {
            margin-top: 10px;
            font-size: 13px;
            color: rgba(0, 0, 0, 0.6);
        }
    </style>
</html>
