<!DOCTYPE html>

<html lang="en">
    <head>
        <title>Results</title>
    </head>
    <body>
        <script src="./scripts/wiki.js"></script>
        <!-- <script src="/Template:Ionis_Paris/scripts/wiki?action=raw&ctype=text/javascript"></script> -->

        <div class="navigation"></div>
        <div class="firstView">
            <div class="container firstViewTitle">Results</div>
        </div>
        <div></div>
        <div class="container">
            <div class="textContent mt-16 addPadding">
                <p>
                    The aim of our project BacTail is to create tailor-made and engineered bacteria to specifically
                    recognize, bind and lyse pathogenic bacteria, then self-destroy, using the powerful tools of
                    synthetic biology. For the experimental approach, we divided our project into 3 distinct axes for
                    for the laboratory:
                </p>
            </div>
            <div class="containerCard" x-data="{}">
                <div class="Card" target-id="phase1">
                    <img :src="window.LINKS.designProjectPhase1.url" class="imgCard" />
                    <div class="textCard1 projectCard">
                        <b>Phase 1</b><br />
                        Target and bind to the bacteria of interest
                    </div>
                </div>
                <div class="Card" target-id="phase2">
                    <img :src="window.LINKS.designProjectPhase2.url" class="imgCard" />
                    <div class="textCard2 projectCard">
                        <b>Phase 2</b><br />
                        Lyse the target bacteria using antimicrobial peptides
                    </div>
                </div>
                <div class="Card" target-id="phase3">
                    <img :src="window.LINKS.designProjectPhase3.url" class="imgCard" />
                    <div class="textCard2 projectCard">
                        <b>Phase 3</b><br />
                        Incorporate of a Kill-switch system in our chassis to induce its self-destruction
                    </div>
                </div>
            </div>
            <div class="check mt-16">
                <i class="checkIcon" data-feather="check-circle"></i>
                <div>
                    <b>The properties of our experimental approach are as follows:</b>
                </div>
            </div>
            <div class="textContent mt-4 ml-16 addPadding">
                <p>
                    All the steps cited above have been developed as one unique system adapted to our project – yet,
                    these steps can be used individually and eventually be re-adapted for other application areas.
                </p>
            </div>
            <div class="textContent mt-4 ml-16 addPadding">
                <p>
                    All our phases are designed for <b><i>Escherichia coli K12</i></b> since we are working under
                    <b>Class 1 Biosafety level</b>. Yet, the ultimate goal is to create a template of this system that
                    can be specifically applied against different bacterial strains. Specific bacteriophages have
                    already been identified to target pathogenic bacteria such as <b><i>Salmonella</i></b
                    >, <b><i>Listeria</i></b
                    >, <b><i>Mycobacterium tuberculosis</i></b
                    >, <b><i>Pseudomonas</i></b> or <b><i>Escherichia coli</i></b
                    >. For example, the PP01 phage was shown to target the shiga-toxin producing
                    <b><i>E. coli</i></b> strain<sup>1</sup>.
                </p>
            </div>
            <div class="textContent mt-4 ml-16 addPadding">
                <p>
                    Therefore, we created a
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Software'">software</a> to specifically
                    select and customize a BioBrick to recognize the desired target bacteria for phase 1.
                </p>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Further detail on our experimental procedures and protocols can be found here:
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Notebook'">Procedures</a> and
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Project/Protocols'">Protocols</a>.
                </p>
            </div>
            <div class="resultTitle mt-16" id="phase1">Phase 1. Targeting and binding to the target bacteria</div>
            <div class="resultTitle1 mt-16">BioBrick and primer design of the fragments for phase 1</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The gene product 37 (gp37) is a critical component for the phase I of the project BacTail. To find
                    out more about the scientific background and
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Design'">design of Phase I</a>.<br /><br />

                    The aim for the first phase is to express the gp37 on the surface of our chassis Escherichia coli
                    (<i>E.coli</i>) K12 thanks to the co-expressed genes Lpp-OmpA. The following BioBrick (<b
                        >Figure 1</b
                    >) was created and divided into 4 different fragments to ease the assembly step with the plasmid –
                    for clarity, the fragments are summarized in <b>Figure 2B</b>.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results1.url" />
            <div class="textContent mt-6 note">
                <b>Figure 1: DNA sequence of BioBrick 1 of phase 1</b> | All the parts to express gp37 on the surface of
                our chassis: Promoter (BBa_J23104), Ribosome binding site (RBS - BBa_B0030), Co-expressed lipoprotein
                with the outer membrane protein A (Lpp-OmpA - BBa_K1694002), Short linker (Gly-Gly-Ser-Gly) to fuse the
                proteins (BBa_K243004), TEV cleavage site (BBa_J18918), 10xHis-tag (BBa_K844000), Red fluorescent
                protein (RFP - BBa_E1010), Gene product (gp) 37 (BBa_K3409000), Double-terminator (BBa_B0015), Promoter
                medium (BBa_J23107), Ribosome binding site (RBS - BBa_B0032), chaperone protein gp38 (BBa_K3409001),
                chaperone protein 57A (BBa_K3409002), Terminator (BBa_B1005) | Created with Benchling
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results2.url" />
            <div class="textContent mt-6 note">
                <b>Figure 2: BioBrick 1 divided into four distinct fragments | A.</b> The expected assembled plasmid
                with Fragments 1,2,3,4 and the backbone | <b>B.</b> A table giving the detailed composition of the four
                fragments of the first phase of BacTail.
            </div>
            <div class="resultTitle1 mt-16">Polymer chain reaction (PCR) & gel migrations</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Once received, the synthetized fragments are resuspended and amplified via polymerase chain reaction
                    (PCR) to obtain a higher quantity of DNA. Every PCR sample is verified by gel electrophoresis to
                    analyze the PCR results. The first time PCR has been performed (<b>Figure 3A</b>) for Fragment
                    (Frag.) 1, Frag. 2, Frag. 3 and Frag. 4, no bands are observable. After some troubleshooting and
                    verifications, new primers need to be ordered to amplify Fragments 1, 2, 3, and 4 again (<b
                        >Figure 3B</b
                    >). However, Fragment 1 required to be amplified in two steps (<b>Figure 3B</b> – Frag.1 & Frag. 1
                    Bis). Due to its complexity, Fragment 1 is synthetized again. Eventually, with the new version of
                    Fragment 1 and the optimized primers the appropriate bands for Frag. 1 (1218 bp), Frag. 2 (1541 bp),
                    Frag. 3 (1669 bp) and Frag. 4 (845 bp) are at the expected size (<b>Figure 3C</b>).
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results3.url" />
            <div class="textContent mt-6 note">
                <b>Figure 3: Gel migration results of Fragment 1,2,3, and 4 with optimization. | A.</b> 1kb+ DNA ladder
                was used for our gels <b>A.</b> First PCR amplification of Fragment 1,2,3 and 4. <b>B.</b> Second round
                of PCR amplification of Fragment 1,2,3, and 4 <b>C.</b> Third and last round with optimized conditions
                of PCR amplification of Fragment 1,2,3, and 4 <b>D.</b> Expected results generated after PCR simulation
                with SnapGene.
            </div>
            <div class="resultTitle1 mt-16">Assembly step</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The four fragments purified in the previous step (see above) sequences have been assembled into the
                    plasmid pSB1A3 using the NEBuilder Hifi Dna Assembly Master Mix. This technique is a molecular
                    cloning method used to join multiple DNA fragments in a single, isothermal reaction2. The fragments
                    were designed in such a way that adjacent fragments have identical sequences on the ends, called
                    overhangs, see <b>Figure 4</b>. Overhangs are added during the synthesis of the different fragments.
                    However, overhang sequences can be added via PCR with primers containing a 5‘ end identical to the
                    adjacent fragment and a 3‘ end that anneals to the target sequence. In our case, the primers are 30
                    base pairs (bp) long, where the first 15 bp match the end of the adjacent fragment and the last 15
                    bp annealed to the target sequence.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results4.url" />
            <div class="textContent mt-6 note">
                <b>Figure 4: Schema of the HiFi Assembly reaction</b> | Created with BioRender.com
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results5.url" />
            <div class="textContent mt-6 note">
                <b>Figure 5: Expected results of the Hifi Assembly of the 4 fragments and the plasmid pSB1A3 | A.</b>
                The expected assembled plasmid with Fragments 1,2,3,4 and the backbone | created with SnapGene
            </div>
            <div class="resultTitle2 mt-16"><b>Conclusion</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Due to difficulties encountered during the experimental procedures and time limitation, we decided
                    to change our verification technique and ordered a new DNA fragment. More information on this in
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Engineering'">Engineering Success</a>.
                </p>
            </div>
            <div class="resultTitle1 mt-16">New BioBrick – Version 2: without RFP</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Due to difficulties encountered during the experimental procedures and time limitation, we decided
                    to change our verification technique and ordered a new DNA fragment. More information on this in
                    Engineering Success.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results6.url" />
            <div class="textContent mt-6 note">
                <b>Figure 6: New DNA sequence of BioBrick 1 version 2 (V2) of phase 1 |</b> All the parts to express
                gp37 on the surface of our chassis: Promoter (BBa_J23104), Ribosome binding site (RBS - BBa_B0030),
                Co-expressed lipoprotein with the outer membrane protein A (Lpp-OmpA - BBa_K1694002), Short linker
                (Gly-Gly-Ser-Gly) to fuse the proteins (BBa_K243004), TEV cleavage site (BBa_J18918), 10xHis-tag
                (BBa_K844000), Gene product (gp) 37 (BBa_K3409000), Double-terminator (BBa_B0015),Promoter medium
                (BBa_J23107), Ribosome binding site (RBS - BBa_B0032), chaperone protein gp38 (BBa_K3409001), chaperone
                protein 57A (BBa_K3409002), Terminator (BBa_B1005) | Created with Benchling
            </div>
            <div class="resultTitle1 mt-16">Revelation of the presence of gp37 via immunofluorescence labelling</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The aim was to detect the receptor binding protein gp37 expressed and directed to the membrane
                    thanks to the co-expressed Lpp-OmpA genes. This is achieved using fused His-tag placed after the
                    OmpA and before the gp37. Thus, the His-tag was targeted with a primary monoclonal anti-his antibody
                    conjugated to a fluorescence marker, fluorescein isothiocyanate (FITC), which excitation and
                    emission spectrum peak has a wavelengths of approximately 495 nm and 519 nm<sup>6</sup>, detectable
                    via fluorescence spectrophotometry.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results7.url" />
            <div class="textContent mt-6 note">
                <b>Figure 7: Expected results of the immunofluorescence labelling | 1st possibility:</b> In the case of
                binding of the monoclonal anti-his antibody to the his-tag, a signal is detected at 519 nm.
                <b>2nd possibility:</b> No attachment of the antibody and therefore no signal is detected at 519 nm |
                Created with BioRender.com
            </div>
            <div class="resultTitle2 mt-16">
                <b>Results of the validation assay of phase I</b>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The aim of this assay is to target the his-tag fused to the receptor binding protein gp37, which
                    should be directed to the outer membrane of our transformed bacteria, via a monoclonal anti-his
                    antibody conjugated to fluorescein isothiocyanate (FITC). Thus, after the immunolabelling, the
                    fluorescence is measured at excitation wavelength of 497 nm and emission wavelength of 519 nm.<br /><br />

                    The controls for this assay are bacteria that have been transformed with the pSB1A3 containing the
                    RFP only and that undergo the same treatment as our samples. The samples are bacteria containing the
                    plasmid construct of phase I within the his-tag fused to the receptor binding protein gp37.
                    Additionally, samples and controls are treated with and without anti-his antibodies diluted at
                    1/1000 and 1/100. Finally, our samples have been done in triplicate (S1-series, B1-series and B2
                    series).
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results7Bis.url" />
            <div class="textContent mt-6 note">
                <b>Figure 8: Results obtained after plate reading with the fluorescent spectrophotometer</b>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    We can observe that all the negative control samples, which are the samples that are not treated
                    with the anti-his antibody (S1, B1, B2) and the control (C) emit a similar signal around 27000.
                    Samples treated with the anti-his antibody 1/1000 dilution (S1 Ab <sub>1/1000</sub>, B1 Ab
                    <sub>1/1000</sub>, B2 Ab <sub>1/1000</sub> ) and the control that has been treated with the anti-his
                    antibody 1/1000 dilution (C Ab <sub>1/1000</sub>), emit a signal around 27000 as well. Finally,
                    samples treated with the anti-his antibody 1/100 dilution (S1 Ab <sub>1/100</sub>, B1 Ab
                    <sub>1/100</sub> , B2 Ab <sub>1/100</sub> ) emit a signal around 43,000, 38,000, 40,000 restively,
                    and the control that has been treated with the anti-his antibody 1/100 dilution (C Ab
                    <sub>1/100</sub>) emit a signal around 40,000.<br /><br />

                    Thus, S1 Ab <sub>1/100</sub> (43,000) is the only sample of transformed bacteria with the construct
                    of phase I that has a higher signal than its controls (C Ab <sub>1/100</sub> = 40,000 and S1 without
                    antibody = 27,000), which is not concluding enough to suggest that the receptor binding protein is
                    well expressed on the outer membrane of our chassis. However, the fact that we have similar signals
                    for each conditions shows that the measured signals are well due to the presence or absence of the
                    anti-his antibody. However, what this data shows is that the antibody might bind unspecifically to
                    other regions of the bacteria or other components in the reaction tube. This specificity issue could
                    not be analysis in more depth because of a lack of time.<br /><br />

                    After having performed this immunolabelling, we thought that this assay was not enough to confirm
                    the presence or not of the receptor binding protein gp37 on the surface of our chassis. We should
                    rather proceed with lysing the cells and differential centrifugation steps to separate the membrane
                    from the bacterial cytoplasm. Then, both samples should by analyzed by performing a western blot by
                    targeting again the his-tag with an anti-his antibody. This would show if the protein is present in
                    the membrane or in the cytoplasm at all.
                </p>
            </div>

            <div class="resultTitle1 mt-16">
                Experimental perspectives Phase 1 | Targeting and binding to the target bacteria
            </div>
            <div class="textContent mt-4 addPadding">
                <ul>
                    <li>
                        Perform a differential centrifuge to extract the fraction with the membrane the fraction with
                        the cytoplasm and detect the presence of the gp37 fused to the his-tag via Western blot in
                        either fraction.<br /><br />
                    </li>

                    <li>
                        The immunolabelling experiments should be repeated to validate or not the presence of gp37 on
                        the surface of our chassis by optimizing the unspecific binding (Maybe try to use a higher
                        concentration of BSA when performing the blocking step).<br /><br />
                    </li>

                    <li>
                        Perform the experiment with the TEV protease and proceed with the protein purification with the
                        Ni2+ chromatography columns specific to the his-tag.<br /><br />
                    </li>

                    <li>
                        Validate experiments with confocal microscopy to have the possibility to see any signal at the
                        surface of the chassis.<br /><br />
                    </li>

                    <li>Validate the binding of the chassis to the target.<br /><br /></li>
                </ul>
            </div>
            <div class="resultTitle mt-16" id="phase2">Phase 2. Lysis of the target bacteria of interest</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The aim of this phase is to lyse specific target bacteria. Thus, the microcin MccPDI (Proximity
                    Dependent Inhibition) is the critical component in phase 2. The MccPDI gene cluster has been
                    identified in the strain Escherichia coli (E.coli) 25 plasmid (Eberhart et al., 2012 ) for its
                    ability to kill several strains of E. coli. MccPDI targets pathogenic strains as well as Biosafety
                    Level 1 E. coli strains. To find out more about the scientific background and
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Design'">design of Phase II</a>.
                </p>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    For the sake of our project BacTail, it was the necessary to express accurate genes in order to
                    secrete the antimicrobial peptide to specifically lyse the target bacterium. For the construction of
                    our BioBrick, we used the MccPDI gene cluster because its genes are all necessary for the
                    appropriate expression, activity, resistance, and secretion of microcin PDI.
                </p>
            </div>
            <div class="resultTitle1 mt-16">BioBrick and primer design of phase II BioBricks 1,2, and 3</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The organization of the native MccPDI gene cluster found in E. coli 25 helped us design our own
                    BioBricks to enable the engineered bacteria to lyse the bacteria of interest, illustrated below :
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results8.url" />
            <div class="textContent mt-6 note">
                <b>Figure 9: Our constructs for the expression and secretion of McpM | BioBrick 1</b> (BBa_K3409010)
                contains the gene mcpM which when expressed, will lead microcin PDI. This microcin has two N-Ter signal
                peptides which are cleaved when secreted. The gene mcpA also present in this BioBrick encodes for an
                activator of McpM and contains a CaaX protease domain. Finally, the gene mcpI encodes for the immunity
                protein of McpM for the producing strain; <b>BioBrick 2</b> (BBa_K3409011) is coding for the McpD
                protein which acts as a membrane fusion protein creating a channel through the periplasm and connects to
                TolC; <b>BioBrick 3</b> (BBa_K3409012) is coding for the McpB protein which contains transmembrane
                domains as well as nucleotide binding domains which have highly conserved motifs characteristic of the
                ABC transporter superfamily of proteins | Created with Benchling.
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results9.url" />
            <div class="textContent mt-6 note">
                <b>Figure 10: The final construct for the expression and secretion of McpM | A.</b> The expected
                assembled plasmid with BioBricks 1,2,3 and the backbone | <b>B.</b> A table giving the detailed
                composition of the three BioBricks of the second phase of BacTail.
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Concerning the design, the three BioBricks were synthetized for further assembly via IDT. Thus, we
                    decided to add homologous sequences by PCR between the fragments to assemble for
                    <b>Hifi Assembly</b> method (see Hifi Assembly in the Assembly section further down this the page).
                    Additionally, we added BioBrick prefix and suffix enzyme restriction sites to assemble the entire
                    cluster via the <b>BioBrick Assembly</b> method as well (see BioBrick Assembly in the Assembly
                    section further down this the page).
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results10.url" />
            <div class="textContent mt-6 note">
                <b
                    >Figure 11: Schematic representation of the three BioBricks for phase II and their corresponding
                    length (bp) | A.</b
                >
                Primers for the BioBrick Assembly method amplifying on the prefix and suffix region <b>B.</b> Primers
                for the Hifi Assembly method, the primers bind after the prefix and suffix region and add a homologous
                short sequence of the fragment it will be assembled with | Created with BioRender.com
            </div>
            <div class="resultTitle1 mt-16">
                Polymer chain reaction (PCR) and gel migration of BioBrick 1, BioBrick 2, and BioBrick 3
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Once received, the synthetized fragments are resuspended and amplified via polymerase chain reaction
                    (PCR) to obtain a higher quantity of DNA. Every PCR sample is verified by gel electrophoresis to
                    analyze the PCR results. First, the appropriate bands for BioBrick 1 (1.5 kb), BioBrick 2 (1.4 kb),
                    and BioBrick 3 (2.4 kb) amplified with the primers for Hifi Assembly method are present at their
                    expected size (<b>Figure 12A</b>). Then, the appropriate bands for BioBrick 1 (1.5 kb), BioBrick 2
                    (1.4 kb), and BioBrick 3 (2.4 kb) amplified with the primers for BioBrick Assembly method, are
                    present at their expected size on <b>Figure 12B</b>.Finally, BioBrick 2 (1.4 kb) and BioBrick 3 (2.4
                    kb) amplified with the primers for BioBrick Assembly method, are present at their expected size on
                    <b>Figure 12C</b>.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results11.url" />
            <div class="textContent mt-6 note">
                <b>Figure 12: Gel migration results of BioBricks 1, 2 and 3 |</b> A 1kb+ DNA ladder was used for all
                gels A. PCR amplification of BioBrick (BB) 1(1.5 kb) , BioBrick 2 (1.4 kb) and BioBrick 3 (2.4 kb) with
                primers for Hifi Assembly method B. PCR amplification of BioBrick (BB) 1(1.5 kb), BioBrick 2 (1.4 kb)
                and BioBrick 3 (2.4 kb) with primers for BioBrick Assembly method C. Expected results simulated with
                SnapGene containing the 1 kb+ DNA ladder, BioBrick (BB) 1(1.5 kb) , BioBrick 2 (1.4 kb) and BioBrick 3
                (2.4 kb).
            </div>
            <div class="resultTitle1 mt-16">Assembly step</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The three fragments purified in the previous step (see above) are assembled into the plasmid pSB1A3,
                    pSB4A5 or pSB4K5 either using Hifi Assembly or restriction enzyme (RE) digestion and ligation.
                </p>
            </div>
            <div class="resultTitle1 mt-16">A. Hifi DNA Assembly method</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Hifi DNA Assembly method is a molecular cloning technique used to join multiple DNA fragments in a
                    single isothermal reaction<sup>2</sup>. As depicted in
                    <b>Figure 11B</b>, the three BioBrick fragments were designed to add identical end sequences, or
                    overhangs to adjacent fragment with specific primers via PCR.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results12.url" />
            <div class="textContent mt-6 note">
                <b>Figure 13: Hifi Assembly Method of BioBricks 1,2 and 3 |</b> 1. BioBrick fragments and backbone are
                put together for assembly; 2. The end results is the assembled circular vector containing all the
                BioBricks and plasmid backbone; 3. Assembled product is transformed into competent cells | Created with
                BioRender.com
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results13.url" />
            <div class="textContent mt-6 note">
                <b
                    >Figure 14: Transformation after Hifi assembly of Phase 2 – BioBrick 1,2,3 and pSB1A3 | Plate on the
                    left:</b
                >
                Escherichia coli DH5 alpha transformed with Hifi assembled product of Phase 2 (BioBrick 1,2,3 and
                pSB1A3). Plate on the right: Escherichia coli DH5 alpha transformed with the Hifi Assembly positive
                control.
            </div>
            <div class="resultTitle2 mt-16"><b>Conclusion</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    After Hifi Assembly for phase 2, we should obtain a plate with a certain number of white colonies.
                    However, all the colonies we can see on the plate on the left of Figure 6 are red and outnumbered by
                    small colonies present on the positive control, which can be observed on right plate of Figure 6.
                    These results show that the Hifi Assembly for the phase 2 did not worked. The red color of the
                    colonies suggests that the cells were transformed with the plasmid still containing the RFP gene yet
                    not the inserts. This experiment has been repeated 3 times with no concluding outcome. The second
                    strategy has been carried out for the assembly, although it is more time-consuming .
                </p>
            </div>
            <div class="resultTitle1 mt-16">B. Restriction enzyme (RE) digestion & ligation assembly method</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Different workflows have been tested in parallel for the BioBrick Assembly method. One is the
                    <b>2-step workflow</b> where the the ligation is performed in 2 steps as depicted in
                    <b>Figure 16</b>. The other one is the <b>1-step workflow</b> where the ligation is in done in one
                    step.
                </p>
            </div>
            <div class="resultTitle1 mt-16">The 2-step workflow</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    BioBrick 2 and 3 need to be assembled first since BioBrick 1 contains the McpM which is toxic for
                    bacteria (see Design Phase 2). Thus, BioBrick 2 and 3 are not killing bacteria during their
                    transformation. As a test, BioBrick 1 is inserted into pSB1A3 and transformed into competent cells,
                    see <b>Figure 15</b>.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results14.url" />
            <div class="textContent mt-6 note">
                <b>Figure 15: DH5 alpha bacteria transformed with BioBrick 1 in pSB1A3 |</b> Plates on the left are the
                controls (Bacteria transformed only with the plasmid incubated at 30°C and 37°C) and plates on the right
                are bacteria that have been transformed with the ligated product (plasmid + BioBrick 1 incubated at 30°C
                and 37°C).
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Concerning the test of the insertion of BioBrick 1 into the pSB1A3, plates have been incubated at
                    30°C and 37°C degrees. Both plates with the sample (Figure 14 – on the right) containing only
                    BioBrick 1 do not show any colony. On the other side, the control plates (Figure 14 – on the left)
                    displayed a few colonies. This suggests that bacteria was not able to survive when expressing McpM
                    in BioBrick 1. McpM thus seems to be toxic for the bacteria.
                </p>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    To assemble BioBrick 2, 3, and the backbone, BioBrick 2 is digested with EcorI and SpeI, BioBrick 3
                    is digested with XbaI and PstI, and the backbone with EcorI and PstI. The cohesive ends of the cut
                    SpeI and XbaI sites can ligate. This resulted in the incorporation of the BioBrick scar sequence (5'
                    TACTAGAG 3’) between BioBrick 2 and 3.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results15.url" />
            <div class="textContent mt-6 note">
                <b>Figure 16: 2-step BioBrick Assembly method of BioBricks 1,2 and 3 |</b> 1. Restriction (RE)
                digestion: BB2 with EcoRI and SpeI, BB3 with XbaI and PstI, backbone with EcorI and PstI; 2. Ligation;
                3. Bacterial transformation with the ligated product; 4. Miniprep of the transformed bacteria to collect
                the plasmid; 5. RE digestion of the plasmid construction with EcorI and XbaI and BB1 EcorI and SpeI; 6.
                Ligation; 7. Bacterial transformation with the ligated product | Created with BioRender.com
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The first ligation product of the <b>2-step workflow</b> which is BioBrick 2, BioBrick 3, and the
                    plasmid pSB1A3 is validated by colony PCR and RE digestion.
                </p>
            </div>
            <div class="resultTitle2 mt-16"><b>Results of the 2-step workflow</b></div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results16.url" />
            <div class="textContent mt-6 note">
                <b
                    >Figure 17: Transformation of assembled BB2, BB3 and pSB1A3 | A. DH5 alpha plates with different
                    ratio of ligated product of BB2, BB3 and pSB1A3, and the control:</b
                >
                Plate N°1 is the control (Cut plasmid + ligase without the insert); Plate N°2 Ligation 1:2 ratio; Plate
                N°2 Ligation 1:2 ratio; Plate N°3 Ligation ratio 1:5; Plate N°4 Ligation ratio 1:3
                <b>B. Gel migration of verification RE digestions Well N°1, 14:</b> 1kb+ DNA Ladder //
                <b>Well N°2, 3, 4, 5, 6:</b> Colonies 1, 2, 3, 4, 5 of Plasmid pSB1A3, BB2, BB3 ( Ratio 1:5) digested
                with BglI+ NdeI // <b>Well N°7, 8, 9, 10, 11, 12:</b> Colonies 1, 2, 3, 4, 5 of Plasmid pSB1A3, BB2, BB3
                ( Ratio 1:2) digested with BglI+ NdeI // <b>Well N°13:</b> Ligation (20/10/20) pSB1A3, BB2, BB3
                <b
                    >C. Expected results obtained with virtual digestion of plasmid BB2,BB3 construction with
                    SnapGene:</b
                >
                Well N°MW: 1kb+ DNA ladder // Well N°1: Assembled plasmid cut with BglI+ NdeI.
            </div>
            <div class="resultTitle2 mt-16"><b>Conclusion of the 2-step workflow</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    When compared to different ligation ratios, a proportional amount of white colonies can be observed
                    on the plates (Figure 17A) with no colonies counted on the negative control plate. Thus, the
                    transformation seems to have worked. To verify the successful ligation of the plasmid with BB2 and
                    BB3, colonies are tested after miniprep DNA isolation to purify the plasmid of interest. The
                    plasmids purified from the bacterial colonies have been cut with restriction enzymes (RE) BgII and
                    NdeI. The expected bands are depicted on <b>Figure 17C</b>. Sample of lane 2, 4, and 6 ,
                    corresponding to colony N°1, colony N°3, and colony N°5 respectively, seem to have bands at 3.5 kb
                    and 2.4 kb which correspond to the expected size.<br /><br />

                    This workflow was stopped at this step for a matter of time. However, the next step would have been
                    to assembly the obtained plasmid of colonies 1, 3, and 5 with the BioBrick and after confirmation to
                    continue with the verification procedures (Via agar diffusion assays among others).
                </p>
            </div>
            <div class="resultTitle1 mt-16">The 1-step workflow</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    To assemble BioBrick 1, BioBrick 2, BioBrick 3, and the backbone, BioBrick 1 is digested with EcoRI
                    and SpeI, BioBrick 2 is digested with XbaI and SpeI, BioBrick 3 is digested with XbaI and PstI, and
                    the backbone with EcorI and PstI. The cohesive ends of the cut SpeI and XbaI sites can ligate
                    together and as a result a BioBrick scar sequence (5' TACTAGAG 3’) will be between both BioBrick 1
                    and 2, and BioBrick 2 and 3.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results17.url" />
            <div class="textContent mt-6 note">
                <b>Figure 18: 1-step BioBrick Assembly method of BioBricks 1,2 and 3 | </b>All in the same tube: 1. 1st
                Ligation with BioBrick 1 and BioBrick 2; 2. Addition of BioBrick 3 for 2nd Ligation; Addition of the
                plasmid for 3rd ligation. Assembled plasmid can be transformed| Created with BioRender.com
            </div>
            <div class="resultTitle2 mt-16"><b>Results of the 1-step workflow</b></div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results18.url" />
            <div class="textContent mt-6 note">
                <b
                    >Figure 19: 1-step workflow - Transformation of assembled BB1, BB2, BB3 and pSB1A3 | A. DH5 alpha
                    plates with ligated product of BB1, BB2 BB2 and pSB1A3, and the control:</b
                >
                Plate N°1 One-step workflow ligated product with plasmid 39; Plate N°2 One-step workflow ligated product
                with plasmid 31 <b>B. Gel migration of the RE digestions: Well N°1,2,3</b>: 1kb DNA Ladder //
                <b>Well N°2, 3, 4</b>: Colony C1, C2, C3 of plasmid 39 Phase II (miniprep) digested by BglI // Well N°5
                -> 14 : Colony 1 ->10 plasmid 39 Phase II (miniprep) digested by BglI // <b>Well N°15, 16, 17, 18</b>:
                Colony C1, C2, C3, C4 of plasmid 31 phase II (miniprep) digested by NdeI //
                <b>Well N°19, 20, 21, 22</b>: Colony C1, C2, C3, C4 of plasmid 31 phase II (miniprep) digested by SapI
                <b>C. Expected resultsimulated with SnapGene</b>: Well N°MW : 1kb+ DNA Ladder //Well N°1: plasmid 39
                Phase II digested by BglI expected //Well N°2: plasmid 31 Phase II digested by SapI expected.
            </div>
            <div class="resultTitle2 mt-16"><b>Conclusion of the 1-step workflow</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Several white colonies are obtained on plate N°1 and N°2 (<b>Figure 19A</b>) and no colonies are
                    counted on the control (plate not present). Thus, the transformation seems to have worked. Regarding
                    the verification of the ligation, colonies needed to be tested after having completed a miniprep
                    procedure to obtain the plasmid. The extracted plasmids of the colonies have been cut with
                    restriction enzymes (RE) other than the one used for the BioBrick Assembly method. Then, NdeI is
                    used to linearize the plasmid 31. SapI is used to cut twice plasmid 31 to obtain 2 distinct bands at
                    10 kb and 3.3 kb as depicted on Well N°2 <b>Figure 11C</b>. For plasmid 39 we did not have any RE
                    available in our inventory to linearize it. Thus, we only used BglI to cut twice plasmid 39 to
                    obtain 2 distinct bands at 6.7 kb and 1.6 kb as depicted on Well N°1 <b>Figure 19C</b>.

                    <b>For colony 3 of plasmid 39</b>: On lane 4, three bands can be observed: one around 6.7 kb and one
                    around 1.6 kb and a last one unexpected around 3.5 kb. The two first bands correspond to the
                    expected results. Nevertheless, we proceeded with the verification step for the colony 3 of plasmid
                    39.
                </p>
            </div>
            <div class="resultTitle1 mt-16">Verification of the McpM expression</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The aim of this phase is to lyse specific target bacteria through the expression of the microcin
                    McpM. After the assembly of the three BioBricks, the presence of McpM in the introduced plasmid need
                    to be confirmed.
                </p>
            </div>
            <div class="resultTitle1 mt-16">A. Extraction, purification and validation of McpM production</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    To validate the production of McpM, the protein need to be extracted and purified from the cellular
                    culture via Ni2+ chromatography column and further revelation via SDS-Page and staining technique.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results19.url" />
            <div class="textContent mt-6 note">
                <b>Figure 20: Extraction, purification and validation of McpM production |</b> 1. Transformation of the
                competent bacteria with our final construction containing the McpM and its secretion system; 2.
                Overnight growth of the cells; 3. Overnight inoculation of the appropriate clone; 4. Cell lysis; 5.
                Purification with the his-tag with a Ni2+ IMAC column; 6. SDS-Page, analysis of purification via
                staining | Created with BioRender.com
            </div>
            <div class="resultTitle2 mt-16"><b>Expected results</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The McpM size is 8.2 kDa. Therefore, we expect that in the lane of the purified cell lysate
                    (elutions) a band appears at the size of 8.2 kDa. In the first lane, this band should be more
                    intense than lanes where Elution 2 and 3 have been loaded. We also expect to have few bands in the
                    flow-through, corresponding to potential waste material. Regarding the lysate, the same bands as the
                    flow through might appear, plus one at 8.2 kDa corresponding to McpM. Finally, in the lane of the
                    Laemmli no bands should appear.
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results20.url" />
            <div class="textContent mt-6 note">
                <b>Figure 21: Expected results of extraction, purification and validation of McpM production |</b> Well
                1: Ladder, Well 2: Elution tube N°1, Well 3: Elution tube N°2, Well 4: Elution tube N°3, Well 5: Flow
                through, Well 6: Total lysate, Well 7: Laemmli
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results21Bis.url" />
            <div class="textContent mt-6 note">
                <b
                    >Figure 22: SDS-PAGE and Coomassie staining of the purified cell lysate, flow through and total
                    lysate to validate the presence of McpM |</b
                >
                Well 1: Ladder, Well 2: Elution tube N°1, Well 3: Elution tube N°2, Well 4: Elution tube N°3, Well 5:
                Flow through, Well 6: Total lysate.
            </div>
            <div class="resultTitle2 mt-16"><b>Conclusion</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The chromatography was not 100% efficient as the elution bands still contain many proteins (even
                    though the band are less intense than for the total lysate). The molecular weight marker did not
                    migrate very well as the well was a little blocked by polymerized acrylamide towards the top. The
                    10kDa did not migrate correctly and is not horizontal. The bottom of the gel is not perfect, and it
                    is difficult to visualize proteins. A light band can be observed at a weight inferior to 10kDa in
                    the elution 2 lane. This band is expected to be the McpM which is 8.2kDa and which is not
                    necessarily supposed to be present in large amount. However, we cannot be sure of the presence of
                    McpM in well 3 (elution 2).<br /><br />
                    To completely validate the presence of McpM in the purified cell lysate, a Western blot and
                    immunodetection using an anti-His antibody should be performed. This is done by transferring the
                    proteins from the gel to a nitrocellulose membrane and then performing the immunodetection, with a
                    direct rabbit anti-His antibody conjugated to Horseradish Peroxidase, then detecting color using the
                    HRP color detecting reagent. This will allow us to fully confirm that the low-intense band present
                    10kDa corresponds to McpM.<br /><br />
                    For a matter of time and available equipment, we were not able to perform the Western Blot.<br /><br />
                    Instead, we performed an agar diffusion assay to validate the production and activity of McpM (see
                    below) by the clone containing plasmid 39.<br /><br />
                </p>
            </div>
            <div class="resultTitle1 mt-16">B. Agar diffusion assay</div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results21.url" />
            <div class="textContent mt-6 note">
                <b>Figure 23: Agar diffusion assay to validate the secretion of McpM | 1. The day before:</b> Overnight
                inoculation of the Escherichia coli (E.coli) transformed only with the pSB1A3, E.coli DH5 alpha
                transformed with our construction (BB1,BB2,BB3), E.coli DH10B containing the native MccpDI cluster.
                <b>2. The day of the experiment:</b> The E.coli only with the pSB1A3 are lawn on the plate (spread
                culture onto the agar) and 5-10 ul of supernatant of E.coli DH10B, E.coli DH5 alpha (BB1,BB2,BB3) and LB
                only are added on 6-mm ter discs on the plate and placed at 37°C overnight. <b>3. The day after:</b> A
                little circle should appear around the disc of the cultures that expressed mcpM | Created with
                BioRender.com
            </div>
            <div class="resultTitle2 mt-16"><b>Expected results</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The E.coli DH5 alpha (BB1,BB2,BB3) should kill the surrounding plated E.coli DH5 alpha when it
                    expresses McpM. Therefore, they are forming little radiuses around their colony of the white 6-mm
                    filter discs. This observation confirms their ability to kill E.coli DH5 alpha bacteria. This
                    antimicrobial activity is what we are aiming for phase 2 of the BacTail project.
                </p>
            </div>
            <div class="resultTitle2 mt-16"><b>Results</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    We tested in duplicate colony with plasmid 39 assembled with BioBrick 1,2, and 3 (Disc N°2 and N°4),
                    in duplicate colony with plasmid 31 assembled with BioBrick 1,2, and 3 (Disc N°1 and 3), in
                    duplicate E.coli DH10B strain containing the native MccpDI cluster (Disc N°5 and 7) and as a control
                    only LB medium (Disc 6).
                </p>
            </div>
            <img x-data="{}" class="imgColiKaze mt-8 shadow-sm" :src="window.LINKS.results22.url" />
            <div class="textContent mt-6 note">
                <b>Figure 24: Results of the agar diffusion assay |</b> Plate on the left side: Assay N°1 and plate on
                the right: Assay n°2; For both plates: E.coli DH5 alpha transformed with pSB1A3 containing RFP bacterial
                lawn; Disc 1: E.coli DH5 alpha colonies from plasmid 31; Disc 2: E.coli DH5 alpha colonies from plasmid
                39; Disc 3: E.coli DH5 alpha colonies from plasmid 31; Disc 4: E.coli DH5 alpha colonies from plasmid
                39; Disc 5: Positive control - E.coli DH10B strain containing the native MccpDI cluster; Disc 6:
                Negative control- LB medium; Disc 7: Positive control - E.coli DH10B strain containing the native MccpDI
                cluster.
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    To discuss the results, we zoomed on each disc of both assays, summarized in the table below. Two
                    characteristics are taken into consideration during observation: presence or absence of a radius
                    around the disc (due to the excretion of McppPDI microcin) and if some colonies have grown onto the
                    disc or if the disc remains white.
                </p>
            </div>

            <table class="agarTable mt-16 shadowed" id="agarTable">
                <thead>
                    <tr>
                        <td>Disc N° and compound</td>
                        <td>Assay N°1</td>
                        <td>Assay N°2</td>
                        <td>Disc observations</td>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td>1 – plasmid 31</td>
                        <td><img src="https://2020.igem.org/wiki/images/6/60/T--Ionis_Paris--1_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/b/b6/T--Ionis_Paris--1_2.png" /></td>
                        <td>
                            <li>Disc 1 Assay N°1: red colonies have grown on the disc and no radius around the disc</li>
                            <li>Disc 1 Assay N°2: red colonies have grown on the disc and no radius around the disc</li>
                        </td>
                    </tr>
                    <tr>
                        <td>2 – plasmid 39</td>
                        <td><img src="https://2020.igem.org/wiki/images/2/2a/T--Ionis_Paris--2_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/b/bd/T--Ionis_Paris--2_2.png" /></td>
                        <td>
                            <li>Disc 2 Assay N°1: no red colonies have grown on the disc and radius around the disc</li>
                            <li>
                                Disc 2 Assay N°2 (40 uL instead of 20 uL): no red colonies have grown on the disc and
                                radius around the disc
                            </li>
                        </td>
                    </tr>
                    <tr>
                        <td>3 – plasmid 31</td>
                        <td><img src="https://2020.igem.org/wiki/images/1/1a/T--Ionis_Paris--3_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/a/aa/T--Ionis_Paris--3_2.png" /></td>
                        <td>
                            <li>
                                Disc 3 Assay N°1: red colonies grown on the perimeter of the disc and no a radius around
                                the disc
                            </li>
                            <li>
                                Disc 3 Assay N°2: red colonies have grown on the perimeter of the disc and no a radius
                                around the disc
                            </li>
                        </td>
                    </tr>
                    <tr>
                        <td>4 – plasmid 39</td>
                        <td><img src="https://2020.igem.org/wiki/images/6/6b/T--Ionis_Paris--4_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/8/8a/T--Ionis_Paris--4_2.png" /></td>
                        <td>
                            <li>
                                Disc 4 Assay N°1: red colonies have grown on the perimeter of the disc and no a radius
                                around the disc
                            </li>
                            <li>
                                Disc 4 Assay N°2: red colonies have grown on the perimeter of the disc and no a radius
                                around the disc
                            </li>
                        </td>
                    </tr>
                    <tr>
                        <td>5 – Positive control (plasmid original cluster)</td>
                        <td><img src="https://2020.igem.org/wiki/images/c/c9/T--Ionis_Paris--5_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/3/32/T--Ionis_Paris--5_2.png" /></td>
                        <td>
                            <li>
                                Disc 5 Assay N°1: no red colonies have grown on the perimeter of the disc and a radius
                                around the disc
                            </li>
                            <li>Disc 5 Assay N°2: a radius around the disc</li>
                        </td>
                    </tr>
                    <tr>
                        <td>6 – Negative control (LB media only)</td>
                        <td><img src="https://2020.igem.org/wiki/images/1/1b/T--Ionis_Paris--6_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/9/96/T--Ionis_Paris--6_2.png" /></td>
                        <td>
                            <li>
                                Disc 5 Assay N°1: no red colonies have grown on the perimeter of the disc and a radius
                                around the disc
                            </li>
                            <li>Disc 5 Assay N°2: a radius around the disc</li>
                        </td>
                    </tr>
                    <tr>
                        <td>7 – Positive control (plasmid original cluster)</td>
                        <td><img src="https://2020.igem.org/wiki/images/8/84/T--Ionis_Paris--7_1.png" /></td>
                        <td><img src="https://2020.igem.org/wiki/images/e/e8/T--Ionis_Paris--7_2.png" /></td>
                        <td>
                            <li>Disc 7 Asssay N°1 fell –no results</li>
                            <li>
                                Disc 7 Asssay N°2: No red colonies have grown on the disc perimeter and a radius around
                                the disc
                            </li>
                        </td>
                    </tr>
                </tbody>
            </table>

            <div class="resultTitle2 mt-16"><b>Conclusion</b></div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Around the original strains (Disc 5 and 7 - positive controls) a small radius (1mm) is observed, and
                    no red colonies have grown on the disc perimeters, which suggests that the E.coli DH5 alpha of the
                    bacterial lawn have been killed or could not grow in this area of the plate due to the presence of
                    the original MccPDI cluster. On the contrary, around the negative control (Dis 6 – LB medium only),
                    no radius can be observed, and red colonies could grow on the disc, which shows that nothing hinders
                    the E.coli DH5 alpha of the bacterial lawn to grow on this region. Regarding our samples, we
                    performed the agar diffusion assay with the candidates kept after the construction validation (<b
                        >Figure 11B</b
                    >). We could observe on Disc N°2 Assay N°1 (the construction of plasmid 39), that no red colonies
                    have grown on the disc perimeters and a small radius (1 mm) appeared on the disc perimeters.
                    Additionally, a three-time bigger radius (3mm) on the disc N°2 Assay N°1 (construction of plasmid
                    39) could be observed. This difference in radius size can be explained by the higher volume that has
                    been used for Disc N°2 Assay N°2. Finally, this observations shows that the construction of plasmid
                    39 has the same effect than the one of the original MccpDI cluster, meaning to induce the growth of
                    the E.coli DH5 alpha of the bacterial lawn. This suggest that the construction of plasmid 39
                    expresses the McpM responsible for killing E.coli DH5 alpha.
                </p>
            </div>

            <div class="resultTitle1 mt-16">
                Experimental perspectives of Phase 2 | The lysis of the target bacteria
            </div>
            <div class="textContent mt-4 addPadding">
                <p>For the 1-step workflow:</p>
            </div>
            <div class="textContent mt-4 addPadding">
                <ul>
                    <li>
                        A Western BLOT should be performed to further the validation of McpM production and
                        secretion.<br /><br />
                    </li>

                    <li>
                        Regarding the agar diffusion assay, the experiment should be repeated and E.coli ∆ OmpF should
                        be used as bacterial lawn as a control plate.<br /><br />
                    </li>

                    <li>
                        We should continue with the 2-step workflow to add BioBrick 1 to the already assembled BioBrick
                        2, 3 fragments with the plasmid and transform it into competent cells. After transformation, the
                        plasmid needs to be purified to proceed with the verification of the assembly of BB1, BB2, and
                        BB3, which means digestion of the plasmid with an enzyme that cuts it once to linearize it and
                        another pair of enzymes that cut it twice in the region on the insert. Additionally, the plasmid
                        should be cut with XbaI and PstI to perform an insert release. Furthermore, a colony PCR can be
                        performed. After having verified that the plasmid was well assembled, the selected clone will
                        continue with the further verification steps. These steps include the extraction, purification,
                        and validation of McpM production. Finally, an agar diffusion assay should be performed as well
                        for this 2-step workflow.<br /><br />
                    </li>

                    <li>
                        For future step phase 1/phase 2: insert the plasmid of phase I and phase II into the same
                        bacteria to enable recognition and lysis at the same time and see if the expression of gp37
                        enhances the lysis of the bacteria of interest (it should enable our chassis to approach the
                        target and enhance the antimicrobial concentration in proximity of the target).<br /><br />
                    </li>
                </ul>
            </div>
            <div class="resultTitle mt-16" id="phase3">Results of Phase 3. The Kill-Switch</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    In Phase III, the arabinose operon is the key parameter for the regulation of the death inducting
                    gene DNASE I. To find out more about the scientific background and
                    <a x-data="{}" class="link" :href="window.ROOT_URL + '/Design'">design of Phase III</a>.<br /><br />

                    This phase aims at controlling the life cycle of our engineered bacteria through a kill switch
                    system. In presence of L-arabinose enables the synthesis of the endonuclease bpDNaseI I that will
                    destroy the producer (i.e. the bacteria). For this phase, two constructs were designed. Each of them
                    are divided into 2 different fragments (for a matter of cost and assembly).<br /><br />

                    1. The first fragment (BBa_K3519010) contains, among others, the gene coding for the araC regulatory
                    protein and a minimal pBAD promoter (or araBAD promoter) upstream of the second fragment. Through
                    the binding of araC to L-arabinose, araC enables the arrival of the RNA polymerase to the pBAD
                    promoter so the transcription of the gene of the following fragment can occur. To this fragment, the
                    second or third fragment can be attached (Figure 24).<br /><br />

                    2. The second fragment is the one containing the DNASE I gene coding for the bpDNaseI enzyme.
                    Additionally, a his-tag is added for further purifications and analysis.<br /><br />

                    3. The third fragment possesses a gene coding for the mRFP. This fluorescent protein enables the
                    detection of the expression of the gene following pBAD via fluorescence assays. Furthermore, its
                    quantification will enable the validation of this construct.<br /><br />

                    The fragments are summarized in <b>Table 1</b> for clarity.
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/7/7a/T--Ionis_Paris--resultsPhase3img1.png"
            />
            <div class="textContent mt-6 note">
                <b>Figure 25: Representation of the different fragment assembly and the BioBricks they can form |</b>
                Minimal araBAD promoter (BBa_K2442101), Promoter medium (BBa_J23118), Strong RBS (BBa_BB0034), E. coli
                araC coding region (BBa_K2442103), mRFP (BBa_E1010), DNASE I + poly his-tag (BBa_K3519004), Spacer
                (BBa_0040), Terminator (BBa_0015), araC+araBADpromoter (BBa_K3519010), araC+araBADpromoter+mRFP
                (BBa_K3519011), araC+araBADpromoter+DNASE I (BBa_K3519012).
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/3/32/T--Ionis_Paris--resultsPhase3img2.png"
            />
            <div class="textContent mt-6 note">
                <b>Table 1: List of the fragments composing the BioBricks and the different backbones used</b>
            </div>
            <div class="resultTitle1 mt-16">PCR & Gel Migrations</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    For each fragments, BioBrick prefix and suffix sequences are added on both sides. This is done to
                    assemble the fragments and the backbone all together by restriction enzyme digestion followed by
                    ligation (see restriction enzyme digestion and ligation assembly method further down this the page).
                    However, restriction enzymes need more nucleotides to specifically bind and cut. As the restriction
                    sites are located on both ends of the fragment, few nucleotides are added to the primers.(<a
                        class="link"
                        href="https://international.neb.com/tools-and-resources/video-library/restriction-enzyme-digest-protocol-cutting-close-to-dna-end"
                        >Restriction Enzyme Digest Protocol: Cutting Close to DNA End. <i>New England Biolabs</i></a
                    >)<br /><br />

                    The primers and the fragments are synthetized by IDT. Once received, the synthetized fragments are
                    resuspended and amplified via Polymerase Chain Reaction (PCR) to obtain a higher quantity of DNA.
                    The same process is realized for purified circular pSB1A3, pSB4A5 and pSB4K5 plasmids (already in
                    our possession) to amplify their backbones. Each PCR samples is verified on a 1% agarose gel through
                    electrophoresis to verify its size.<br /><br />

                    After some trouble-shouting, we determined the parameters for an optimal PCR (Figure 26). Concerning
                    the amplification of the fragments, a two-steps PCR is performed as the primers used anneal at 72°C.
                    The annealing temperature for the backbone amplification is about 68°C with another set of primer.
                    While a 50 seconds elongation step is sufficient for the amplification of the fragments, a 2 minutes
                    elongation step is required for the backbones as the longest plasmid, pSB4K5 is 3 378 bp long (Table
                    1).
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/2/24/T--Ionis_Paris--resultsPhase3img3.png"
            />
            <div class="textContent mt-6 note">
                <b> Figure 26: PCR Program used for the amplification of fragments and backbones</b>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The previously presented PCR parameters enabled the amplification of each fragments and backbones as
                    shown in Figure X. Throughout our repetitive PCR, we observe that the amplification of the DNASE I
                    is always leading to double bands. However, one of the two bands correspond to the expected size
                    (1061 bp). Therefore, this band is cut and purified to obtain the fragment bpDNaseI. The same
                    phenomenon is observed for the backbones. One band corresponds to the amplified backbone size
                    (pSB1A3: 2155 bp, pSB4A5: 3395 bp, pSB4K5: 4489 bp) and the other to the size of the circular
                    purified plasmid used as DNA template (pSB1A3: 3225 bp, pSB4A5: 4465 bp, pSB4K5: 3419 bp). Thus, two
                    decisions have been taken:<br /><br />
                    First, a gel purification is realized to discard the circular plasmid template.<br /><br />
                    Second, following this purification step, DpnI is used so it can digest the non-PCR products of this
                    amplification: the template DNA. This way, we reduce the chance of transforming our chassis with the
                    original circular plasmid used for backbone amplifications (<a
                        x-data="{}"
                        class="link"
                        :href="window.ROOT_URL + '/Engineering'"
                        >cf. Engineering success</a
                    >.).
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/0/07/T--Ionis_Paris--resultsPhase3img4.png"
            />
            <div class="textContent mt-6 note">
                <b>
                    Figure 27: Gel migration results following a PCR of the fragments “araC+araBAD promoter”, “mRFP” and
                    “bpDNaseI” and the plasmids pSB1A3, pSB4A5 and pSB4K5 | A.</b
                >
                Expected results with SnapGene Gel simulation <b>B.</b> Gel Caption after migration through UV
                transilluminator <b>C.</b> Wells contents and expected sizes.
            </div>
            <div class="resultTitle1 mt-16">Restriction enzyme (RE) digestion ligation assembly method</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The three fragments purified in the previous step (see above) are assembled into plasmid pSB1A3,
                    pSB4A5 and pSB4K5 using restriction enzyme (RE) digestion and ligation (Figure. X). For this
                    assembly:<br /><br />
                    The fragment “araC+araBAD promoter” is digested with EcoRI and SpeI.<br /><br />
                    The fragment “DNASE I” or “mRFP”, depending on the construct, is digested with XbaI and PstI.<br /><br />
                    The backbone is digested with EcoRI and PstI.<br /><br />
                    The cohesive ends of the cut SpeI and XbaI sites can ligate together but a BioBrick scar sequence
                    (5' TACTAGAG 3’) will result from it between the two fragments. Between the digestion and the
                    ligation step, an alkaline phosphatase was added to the digested backbones to dephosphorylate their
                    cuts so they can not re-ligate (<a x-data="{}" class="link" :href="window.ROOT_URL + '/Engineering'"
                        >For more information see paragraph Alkaline Phosphatase in Engineering Success</a
                    >). Once assembled, the recombinant plasmid is used to transform the chassis.
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/5/5f/T--Ionis_Paris--resultsPhase3img5.png"
            />
            <div class="textContent mt-6 note">
                <b>
                    Figure 28: BioBrick Assembly method to assemble the possible constructs to the backbone for Phase
                    III |</b
                >
                1. Restriction digestion (RE) : araC+araBAD promoter with EcoRI and SpeI, DNASE I or mRFP with XbaI and
                PstI, backbone with EcoRI and PstI; 2. Ligation; 3. Bacterial transformation with the ligated product |
                Created with BioRender.com
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    According to the plasmid backbones which are used for the ligation, six different recombinant
                    plasmids can be obtained (Table. 2). The different constructs can be obtained with pSB1A3, pSB4A5
                    and pSB4K5 backbones associated to BBa_K3519012 (araC+araBADpromoter+DNASE I) and BBa_K3519011
                    (araC+araBADpromoter+mRFP).
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/b/b1/T--Ionis_Paris--resultsPhase3img6.png"
            />
            <div class="textContent mt-6 note">
                <b
                    >Table 2. Different recombinant plasmids that can be obtained after BioBrick assembly method for
                    Phase III.</b
                >
            </div>
            <div class="resultTitle1 mt-16">Transformation</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The assembled constructs (see above) are transformed into NEB DH5α chemically competent
                    <i>E. coli</i>. The results are depicted in the following table (Table .3).
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/b/b9/T--Ionis_Paris--resultsPhase3img7.png"
            />
            <div class="textContent mt-6 note">
                <b>Table 3: Results of transformations with the final constructions for Phase III</b>
            </div>
            <div class="resultTitle1 mt-16">Verification Steps</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    To verify that clones obtained after transformation contained the well assembled construction,
                    several validation assays are realized: Colony PCR, Linearization, Insert release, additional double
                    digestion and <i>DNASE I</i> activation.
                </p>
            </div>
            <div class="resultTitle1 mt-16">1. Colony PCR</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The presence or the absence and the size of the inserts can be verified by Colony PCR followed by an
                    electrophoresis on agarose gel. Additionally, each clone used for this technique are plated on petri
                    dishes (Table .4).
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/1/1f/T--Ionis_Paris--resultsPhase3img8.png"
            />
            <div class="textContent mt-6 note">
                <b>Table 4: Pictures of Colony PCR plates.</b>
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    Through several trouble shouting, the optimal program for Colony PCR was found to be the one
                    presented in
                    <b>Figure 30</b>. There, the elongation time is about 2 minutes as the longest construct,
                    araC+araBADpromoter+DNASEI, is 2538bp long. The set of primer used, VF2 and VR, anneal at 62,5°C.
                    They are commonly used primers for Colony PCR.
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/5/53/T--Ionis_Paris--resultsPhase3img9.png"
            />
            <div class="textContent mt-6 note">
                <b> Figure 29: PCR Program used for the Colony PCR </b>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/7/7f/T--Ionis_Paris--resultsPhase3img10.png"
            />
            <div class="textContent mt-6 note">
                <b>
                    Figure 30: Gel migration results following a Colony PCR on the fragments “araC+araBAD promoter+mRFP”
                    and “araC+araBAD promoter+DNaseI” assembled with pSB1A3, pSB4A5 and pSB4K5 backbones | A.</b
                >
                Expected results with SnapGene Gel simulation <b>B.</b> Gel Caption after migration through UV
                transilluminator <b>C.</b> Wells contents and expected sizes.
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The results of this amplification are shown in <b>Figure 31</b>. For the two controls (circular
                    pSB4A5 and circular pSB4K5 plasmids), bands with the expected size can be observed (1350 bp both).
                    Concerning the samples, only wells N°1 and 4 present bands in adequation with the gel simulation
                    (bands at 2759 bp). They respectively correspond to clones 1 and 4 of the pSB1A3+DNASE I plate.
                    These colonies both seem to have a well constructed recombinant plasmid with the
                    araC+araBADpromoter+DNASEI insert but additional verifications are required.
                </p>
            </div>
            <div class="resultTitle1 mt-16">2. Linearization (Single Digestion)</div>
            <div class="textContent mt-4 addPadding">
                <p>
                    A second way to verify if the transformed bacteria contain our recombinant plasmids is by digesting
                    them with restriction enzymes. First, we chose to linearize the purified plasmids by choosing an
                    enzyme that cuts only once. We decided to go with BglI for bacteria with pSB1A3 backbone and NdeI
                    for pSB4A5 and pSB4K5. Both of their restriction sites are situated on the backbone. The results are
                    presented in <b>Figure 32</b>.
                </p>
            </div>
            <img
                class="imgColiKaze mt-8 shadow-sm"
                src="https://2020.igem.org/wiki/images/f/f6/T--Ionis_Paris--resultsPhase3img11.png"
            />
            <div class="textContent mt-6 note">
                <b>Figure 31: Gel migration results following a single restriction enzyme digestion | A.</b> Expected
                results with SnapGene Gel simulation <b>B.</b> Gel Caption after migration through UV transilluminator
                <b>C.</b> Wells contents and expected sizes.
            </div>
            <div class="textContent mt-4 addPadding">
                <p>
                    The three controls (circular pSB1A3, circular pSB4A5 and circular pSB4K5 digested) gave the expected
                    bands (respectively at 3225 bp, 4465 bp and 5712 bp). Concerning the samples, the observed bands for
                    wells N°2, 4, 5, 9, 11, 16, 19 & 21 correspond to the expected bands on the gel simulation. They
                    respectively correspond to:<br /><br />
                    Colonies 1, 3, 4 of the pSB1A3 + DNASE I plate<br /><br />
                    Colony 4 of the pSB1A3 + mRFP plate<br /><br />
                    Colony 2 of the pSB4A5 + DNASE I plate<br /><br />
                    Colony 3 of the pSB4A5 + mRFP plate<br /><br />
                    Colony 1 of the pSB4K5 + DNASE I plate<br /><br />
                    Colony 1 of the pSB4K5 + mRFP plate<br /><br />
                </p>
            </div>

            <div class="resultTitle1 mt-16">3. Insert release (Double Digestion)</div>
            <div class="textContent mt-6">
                <p>
                    The insert release assay consists in a double digestion of the plasmid at both ends of the insert.
                    It enables the separation of the backbone and the insert. Several enzymes possess a restriction
                    sites in the prefix and suffix region of the assembled fragment. But, due to the restriction enzyme
                    digestion and ligation assembly method, most of them present scares preventing another digestion.
                    XbaI and SpeI being only used for the assembly of the two fragments, their associated restriction
                    sites remain intact at both end of the construct. Thus they represent ideal enzymes for this step,
                    whose results are presented in <b>Figure 32</b>.
                </p>
            </div>
            <img
                src="https://2020.igem.org/wiki/images/b/b7/T--Ionis_Paris--resultsPhase3img12.png"
                class="mt-6 imageContent shadowed"
                style="max-height: 600px"
            />
            <div class="textContent note mt-6">
                <b>Figure 32: Gel migration results following an insert release of the clones of Phase III</b>|
                <b>A.</b> Expected results with SnapGene Gel simulation <b>B.</b> Gel Caption after migration through UV
                transilluminator <b>C.</b> Wells contents and expected sizes.
            </div>
            <div class="textContent mt-8">
                <p>
                    The three controls (circular pSB1A3, circular pSB4A5 and circular pSB4K5 digested) gave the expected
                    bands. Concerning the samples, the observed bands for wells N°4, 16, 19 & 21 correspond to the
                    expected bands on the gel simulation. They respectively correspond to:
                </p>
                <p>
                    <li>Colonies 3 of the pSB1A3 + DNASE I plate</li>
                    <li>Colony 3 of the pSB4A5 + mRFP plate</li>
                    <li>Colony 1 of the pSB4K5 + DNASE I plate</li>
                    <li>Colony 1 of the pSB4K5 + mRFP plate</li>
                </p>
            </div>
            <div class="resultTitle1 mt-16">4. Additional Double Digestion</div>
            <div class="textContent mt-6">
                <p>
                    An additional double digestion is performed to validate all the previous verification assays. While
                    the pSB1A3 recombinant plasmids are digested with XbaI and BglI, pSB4A5 and pSB4K5 backbones are
                    digested with XbaI and NdeI (<b>Figure. 33</b>)
                </p>
            </div>
            <img
                src="https://2020.igem.org/wiki/images/1/13/T--Ionis_Paris--resultsPhase3img13.png"
                class="imageContent mt-6"
                style="max-height: 600px"
            />
            <div class="textContent mt-6 note">
                <b
                    >Figure 33: Gel migration results following the additional insert release of the selected clones of
                    Phase III</b
                >| <b>A.</b> Expected results with SnapGene Gel simulation <b>B.</b> Gel Caption after migration through
                UV transilluminator <b>C.</b> Wells contents and expected sizes
            </div>
            <div class="textContent mt-8">
                <p>
                    The three controls (circular pSB1A3, circular pSB4A5 and circular pSB4K5 digested) gave the expected
                    bands. Concerning the samples, the observed bands for wells N°6, 7, 9 & 10 correspond to the
                    expected bands on the gel simulation. They respectively correspond to:
                </p>
                <p>
                    <li>Colonies 2 of the pSB4A5 + DNASE I plate</li>
                    <li>Colony 3 of the pSB4A5 + mRFP plate</li>
                    <li>Colony 1 of the pSB4K5 + DNASE I plate</li>
                    <li>Colony 1 of the pSB4K5 + mRFP plate</li>
                </p>
            </div>
            <div class="resultTitle1 mt-16">5. DNASE I Activation</div>
            <div class="textContent mt-6">
                <p>
                    Now that we have identified few possible recombinant plasmids containing the <i>DNASEI</i> gene, its
                    expression can be validated via addition of L-arabinose. As the presence of L-arabinose is supposed
                    to activate the kill switch system, bacteria are supposed to produce bpDNaseI, leading to their own
                    destruction (see
                    <a x-data="{}" :href="window.ROOT_URL + '/Phases/phase3'" target="_blank" rel="noopener noreferrer"
                        >Design Phase III</a
                    >). Thus, liquid cultures with antibiotics were incubated with clones presenting at least one good
                    result among the previously described verification assays. In a first step, initial OD<sub>600</sub>
                    of each sample is determined through spectrophotometry. Then, L-arabinose is added and variations of
                    the OD<sub>600</sub> is studied 1h30 later. Following this, even more L-arabinose is added and the
                    OD<sub>600</sub> is measured one last time 1h30 after. A negative control corresponding to
                    untransformed DH5α <i>E. coli</i> was realized but without antibiotic. The results are presented in
                    the following table:
                </p>
            </div>
            <img
                src="https://2020.igem.org/wiki/images/3/30/T--Ionis_Paris--resultsPhase3img14.png"
                class="imageContent mt-8"
            />
            <div class="textContent note mt-6">
                <b
                    >Table 5: OD<sub>600</sub> variations following addition of L-arabinose in potential
                    <i>DNASEI</i> recombinant bacteria</b
                >
            </div>
            <div class="textContent mt-8">
                <p>
                    Globally, OD<sub>600</sub> of each sample was increasing through the experiment similarly to the
                    negative control except for the clone 1 of the pSB4K5 + DNASE I plate. This sample showed a
                    decreased OD following the addition of 100mg of L-arabinose. This information strongly suggest that
                    this clone has the well constructed plasmid.
                </p>
            </div>
            <div class="resultTitle2 mt-16">Conclusion</div>
            <img
                src="https://2020.igem.org/wiki/images/4/4f/T--Ionis_Paris--resultsPhase3img15.png"
                class="imageContent mt-6"
                style="max-height: 400px"
            />
            <div class="note textContent mt-6">
                <b>Table 6</b>: Summary table of clones presenting good results for at least on verification experiment
            </div>
            <div class="textContent mt-8">
                <p>
                    Several clones present good results for the Colony PCR, Linearization, Insert release, Additional
                    double digestion and DNASEI Activation assays. None of them is good for each of them. Nevertheless,
                    the clone 1 of the pSB4K5 + DNASE I plate presents good results for the three digestion assays in
                    addition to the <i>DNASE I</i> Activation assay. These data suggest that this clone possess the
                    right recombinant plasmid. Due to a lack of time, we can not proceed to further investigations. But
                    in the future, we would like to send the clones showing good results for at least two experiments to
                    be sequenced. This will enable us to get the precise DNA sequence of their plasmids.
                </p>
            </div>
            <div class="resultTitle2 mt-16">Perspectives</div>
            <div class="textContent mt-6">
                <p>
                    Once the recombinant plasmids identified and purified, we will transform those containing the
                    <i>mRFP</i>
                    gene into BL21 cells. This will help us determine the optimum arabinose concentration required for
                    maximum induction by the promoter using fluorescence.
                </p>
                <p>
                    Next, recombinant <i>DNASEI plasmid</i> will be transformed into BL21 cells. These cells will be
                    stimulated with the determined appropriate arabinose concentration and cell death will be measured
                    by monitoring the OD of the culture which is supposed to diminish. In addition, the culture will be
                    plated at regular intervals to determine the CFU. Then, DNA from such cultures will be isolated and
                    run on an agarose gel to check for any intact DNA prevailing.
                </p>
                <p>
                    Finally, the bpDNaseI will be overexpressed and will be purified, thanks to its poly his tag
                    sequence, and will be incubated with several DNA concentrations for different time periods for its
                    in-vitro characterization. Thus, DNA degrading efficiency of the enzyme will be determined.
                </p>
            </div>
        </div>
        <div class="references">
            <div class="container">
                <hr />
                <div class="referencesText mb-6">
                    PHASE1<br /><br />
                    [1] Yoichi, M., Abe, M., Miyanaga, K., Unno, H., & Tanji, Y. (2005). Alteration of tail fiberprotein
                    gp38 enables T2 phage to infect Escherichia coli O157: H7. Journal ofbiotechnology, 115(1),
                    101-107.<br /><br />

                    [2] Cas, S. DNA Assembly & Synthetic Biology.<br />
                    https://cdn-api.swapcard.com/public/files/2326aec54f1043499e31fdfc78562e2d.pdf<br /><br />

                    [3] Waugh D. S. (2011). An overview of enzymatic reagents for the removal of affinity tags. Protein
                    expression and purification, 80(2), 283–293.<br />
                    https://doi.org/10.1016/j.pep.2011.08.005<br /><br />

                    [4] Francisco, J. A., Campbell, R., Iverson, B. L., & Georgiou, G. (1993). Production and
                    fluorescence-activated cell sorting of Escherichia coli expressing a functional antibody fragment on
                    the external surface. Proceedings of the National Academy of Sciences of the United States of
                    America, 90(22), 10444–10448.<br />
                    https://doi.org/10.1073/pnas.90.22.10444<br /><br />

                    [5] Nie, D., Hu, Y., Chen, Z. et al. Outer membrane protein A (OmpA) as a potential therapeutic
                    target for Acinetobacter baumannii infection. J Biomed Sci 27, 26 (2020).<br />
                    https://doi.org/10.1186/s12929-020-0617-7<br /><br />

                    [6] The, T. H., & Feltkamp, T. E. (1970). Conjugation of fluorescein isothiocyanate to antibodies.
                    I. Experiments on the conditions of conjugation. Immunology, 18(6), 865–873.<br /><br />
                </div>
            </div>
        </div>
        <footer></footer>
    </body>
    <style>
        html,
        body {
            background-color: #fafafa !important;
        }

        .firstViewTitle {
            color: #8d1f24;
            text-shadow: 0px 10px 20px rgba(141, 31, 37, 0.3);
            font-size: 50px;
            text-transform: uppercase;
            font-weight: 800;
            height: 150px;
            display: flex;
            align-items: flex-end;
        }

        .textContent {
            max-width: 900px;
            font-size: 15px;
        }

        .textContent p {
            margin-bottom: 5px;
        }

        .textContent p:last-child {
            margin-bottom: 0;
        }

        .containerCard {
            display: flex;
            gap: 30px;
            margin-top: 50px;
        }

        .imageContent {
            max-width: 900px;
            width: 100%;
            object-fit: contain;
            padding: 10px;
            background-color: white;
            border-radius: 10px;
            overflow: hidden;
        }

        .Card {
            background-color: white;
            border-radius: 20px;
            -webkit-box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            -moz-box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.03);
            border: 1px solid rgba(0, 0, 0, 0.03);
            height: 300px;
            max-width: 250px;
            flex-grow: 1;
            opacity: 1;
            position: relative;
            display: flex;
            justify-content: center;
            align-items: center;
            transform: scale(1);
            cursor: pointer;
            transition: transform 0.3s;
        }

        .textCard1 {
            position: absolute;
            bottom: 30px;
            margin: 10px;
            text-align: center;
            color: #8d1f24;
        }

        .textCard2 {
            position: absolute;
            bottom: 5px;
            margin: 10px;
            text-align: center;
            color: #8d1f24;
        }

        .imgCard {
            position: absolute;
            bottom: 120px;
            height: 150px;
        }

        .Card:hover {
            transform: scale(1.1);
        }

        .check {
            display: flex;
            align-items: center;
            color: #8d1f24;
        }

        .checkIcon {
            height: 40px;
            width: 40px;
            margin-right: 30px;
            color: #8d1f24;
        }

        .resultTitle {
            color: #8d1f24;
            background-color: rgba(141, 31, 37, 0.1);
            display: inline-block;
            font-weight: 800;
            font-size: 25px;
            line-height: 1;
            padding: 10px 20px;
            border-radius: 10px;
            margin-top: 80px;
        }

        .resultTitle1 {
            color: #8d1f24;
            font-size: 15px;
            font-weight: 800;
        }

        .resultTitle2 {
            font-size: 15px;
            font-weight: 800;
        }

        .imgColiKaze {
            max-width: 900px;
            width: 100%;
            background-color: white;
            border-radius: 15px;
            padding: 10px 10px;
        }

        .note {
            font-size: 13px;
        }

        .references {
            margin-top: 60px;
        }

        .referencesText {
            margin-top: 10px;
            font-size: 13px;
            color: rgba(0, 0, 0, 0.6);
        }

        .container a {
            color: #8d1f24 !important;
            text-decoration: none !important;
        }

        .container a:hover {
            color: #8d1f24 !important;
            text-decoration: underline !important;
        }

        .agarTable {
            background-color: white;
            border-radius: 10px;
            overflow: hidden;
        }

        .agarTable thead {
            font-size: 800;
        }

        .agarTable td {
            padding: 10px;
            border: 1px solid rgb(229, 229, 229);
        }

        .agarTable img {
            max-width: 100%;
            max-height: 200px;
            object-fit: contain;
        }
    </style>
</html>
